---
# jekyll prompt
---
<html>
  <head>
    <meta name="generator"
    content="HTML Tidy for HTML5 (experimental) for Windows https://github.com/w3c/tidy-html5/tree/c63cc39" />
    <meta content="text/html; charset=UTF-8" http-equiv="content-type" />
  <link rel="stylesheet" href="fhir.css"/>
    <style type="text/css">
@import
url(&#39;https://themes.googleusercontent.com/fonts/css?kit=MSSLfUayeNh9PW3ng9UWrirlBcGJ7N4dBtR9TI-y1vHA6mILqzkelCs0stqH5so2&#39;);ol.lst-kix_gzjq6lzg1wwb-2.start{counter-reset:lst-ctn-kix_gzjq6lzg1wwb-2
0}.lst-kix_9hu4a59rs9m9-3&gt;li:before{content:&quot;\0025cf  &quot;}.lst-kix_9hu4a59rs9m9-4&gt;li:before{content:&quot;o 
&quot;}.lst-kix_9hu4a59rs9m9-6&gt;li:before{content:&quot;\0025cf 
&quot;}ol.lst-kix_oai74q1k79ui-2.start{counter-reset:lst-ctn-kix_oai74q1k79ui-2
0}.lst-kix_9hu4a59rs9m9-5&gt;li:before{content:&quot;\0025aa  &quot;}.lst-kix_9hu4a59rs9m9-7&gt;li:before{content:&quot;o 
&quot;}.lst-kix_list_4-3&gt;li{counter-increment:lst-ctn-kix_list_4-3}ul.lst-kix_jphjql4uq7tq-0{list-style-type:none}ul.lst-kix_jphjql4uq7tq-1{list-style-type:none}.lst-kix_nua3ue1dzck6-7&gt;li{counter-increment:lst-ctn-kix_nua3ue1dzck6-7}.lst-kix_9hu4a59rs9m9-8&gt;li:before{content:&quot;\0025aa
 &quot;}ol.lst-kix_list_3-7.start{counter-reset:lst-ctn-kix_list_3-7
0}ul.lst-kix_jphjql4uq7tq-8{list-style-type:none}ul.lst-kix_jphjql4uq7tq-6{list-style-type:none}ul.lst-kix_jphjql4uq7tq-7{list-style-type:none}.lst-kix_list_3-2&gt;li{counter-increment:lst-ctn-kix_list_3-2}ul.lst-kix_jphjql4uq7tq-4{list-style-type:none}ul.lst-kix_jphjql4uq7tq-5{list-style-type:none}ul.lst-kix_jphjql4uq7tq-2{list-style-type:none}ul.lst-kix_jphjql4uq7tq-3{list-style-type:none}.lst-kix_list_5-0&gt;li:before{content:&quot;\0025cf
 &quot;}.lst-kix_list_5-3&gt;li:before{content:&quot;\0025cf  &quot;}.lst-kix_list_5-2&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_list_5-1&gt;li:before{content:&quot;\0025cb  &quot;}.lst-kix_list_5-7&gt;li:before{content:&quot;\0025cb 
&quot;}ul.lst-kix_list_8-4{list-style-type:none}ul.lst-kix_list_8-5{list-style-type:none}.lst-kix_list_5-6&gt;li:before{content:&quot;\0025cf
 &quot;}.lst-kix_list_5-8&gt;li:before{content:&quot;\0025a0 
&quot;}ul.lst-kix_list_8-2{list-style-type:none}ul.lst-kix_list_8-3{list-style-type:none}ul.lst-kix_list_8-8{list-style-type:none}.lst-kix_gzjq6lzg1wwb-8&gt;li{counter-increment:lst-ctn-kix_gzjq6lzg1wwb-8}ul.lst-kix_list_8-6{list-style-type:none}ul.lst-kix_list_8-7{list-style-type:none}.lst-kix_list_5-4&gt;li:before{content:&quot;\0025cb
 &quot;}.lst-kix_list_5-5&gt;li:before{content:&quot;\0025a0 
&quot;}ul.lst-kix_list_8-0{list-style-type:none}ul.lst-kix_list_8-1{list-style-type:none}.lst-kix_list_6-1&gt;li:before{content:&quot;o
 &quot;}.lst-kix_list_6-3&gt;li:before{content:&quot;\0025cf  &quot;}.lst-kix_list_6-0&gt;li:before{content:&quot;\0025cf 
&quot;}.lst-kix_list_6-4&gt;li:before{content:&quot;o 
&quot;}.lst-kix_list_3-6&gt;li{counter-increment:lst-ctn-kix_list_3-6}.lst-kix_list_6-2&gt;li:before{content:&quot;\0025aa 
&quot;}ol.lst-kix_gzjq6lzg1wwb-7.start{counter-reset:lst-ctn-kix_gzjq6lzg1wwb-7
0}ol.lst-kix_list_3-2.start{counter-reset:lst-ctn-kix_list_3-2
0}ul.lst-kix_b1xgmn9bjew7-8{list-style-type:none}ul.lst-kix_b1xgmn9bjew7-7{list-style-type:none}.lst-kix_list_6-8&gt;li:before{content:&quot;\0025aa
 &quot;}ul.lst-kix_b1xgmn9bjew7-6{list-style-type:none}ul.lst-kix_b1xgmn9bjew7-5{list-style-type:none}ul.lst-kix_b1xgmn9bjew7-4{list-style-type:none}ul.lst-kix_b1xgmn9bjew7-3{list-style-type:none}.lst-kix_list_6-5&gt;li:before{content:&quot;\0025aa
 &quot;}.lst-kix_list_6-7&gt;li:before{content:&quot;o 
&quot;}ul.lst-kix_b1xgmn9bjew7-2{list-style-type:none}ul.lst-kix_b1xgmn9bjew7-1{list-style-type:none}ul.lst-kix_b1xgmn9bjew7-0{list-style-type:none}.lst-kix_list_6-6&gt;li:before{content:&quot;\0025cf
 &quot;}.lst-kix_list_7-4&gt;li:before{content:&quot;\0025a0  &quot;}.lst-kix_list_7-6&gt;li:before{content:&quot;\0025cf 
&quot;}ul.lst-kix_9hu4a59rs9m9-8{list-style-type:none}.lst-kix_list_7-2&gt;li:before{content:&quot;\0025c6 
&quot;}ul.lst-kix_9hu4a59rs9m9-4{list-style-type:none}ul.lst-kix_9hu4a59rs9m9-5{list-style-type:none}ul.lst-kix_9hu4a59rs9m9-6{list-style-type:none}ul.lst-kix_9hu4a59rs9m9-7{list-style-type:none}ul.lst-kix_9hu4a59rs9m9-0{list-style-type:none}ul.lst-kix_9hu4a59rs9m9-1{list-style-type:none}ul.lst-kix_9hu4a59rs9m9-2{list-style-type:none}ul.lst-kix_9hu4a59rs9m9-3{list-style-type:none}ol.lst-kix_list_4-6.start{counter-reset:lst-ctn-kix_list_4-6
0}ol.lst-kix_list_3-0.start{counter-reset:lst-ctn-kix_list_3-0
0}.lst-kix_oai74q1k79ui-2&gt;li{counter-increment:lst-ctn-kix_oai74q1k79ui-2}.lst-kix_list_7-8&gt;li:before{content:&quot;\0025c6 
&quot;}.lst-kix_list_4-7&gt;li{counter-increment:lst-ctn-kix_list_4-7}ol.lst-kix_oai74q1k79ui-4.start{counter-reset:lst-ctn-kix_oai74q1k79ui-4
0}.lst-kix_list_4-1&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_list_4-1,lower-latin) &quot;)
&quot;}.lst-kix_list_4-3&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_list_4-3,decimal) &quot;.
&quot;}.lst-kix_list_4-5&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_list_4-5,lower-roman) &quot;.
&quot;}.lst-kix_list_3-5&gt;li{counter-increment:lst-ctn-kix_list_3-5}ol.lst-kix_list_4-4.start{counter-reset:lst-ctn-kix_list_4-4
0}ol.lst-kix_nua3ue1dzck6-1.start{counter-reset:lst-ctn-kix_nua3ue1dzck6-1
0}ol.lst-kix_oai74q1k79ui-7.start{counter-reset:lst-ctn-kix_oai74q1k79ui-7
0}.lst-kix_nua3ue1dzck6-3&gt;li{counter-increment:lst-ctn-kix_nua3ue1dzck6-3}.lst-kix_9hu4a59rs9m9-1&gt;li:before{content:&quot;o 
&quot;}.lst-kix_gzjq6lzg1wwb-4&gt;li{counter-increment:lst-ctn-kix_gzjq6lzg1wwb-4}.lst-kix_list_4-2&gt;li{counter-increment:lst-ctn-kix_list_4-2}ol.lst-kix_list_3-1{list-style-type:none}ol.lst-kix_nua3ue1dzck6-3.start{counter-reset:lst-ctn-kix_nua3ue1dzck6-3
0}ol.lst-kix_list_3-2{list-style-type:none}ol.lst-kix_list_3-3{list-style-type:none}ol.lst-kix_list_3-4.start{counter-reset:lst-ctn-kix_list_3-4
0}ol.lst-kix_list_3-4{list-style-type:none}ol.lst-kix_list_3-0{list-style-type:none}.lst-kix_list_3-0&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_3-0,decimal) &quot;.
&quot;}ul.lst-kix_list_5-7{list-style-type:none}ul.lst-kix_list_5-8{list-style-type:none}.lst-kix_gzjq6lzg1wwb-5&gt;li{counter-increment:lst-ctn-kix_gzjq6lzg1wwb-5}ul.lst-kix_list_5-5{list-style-type:none}ul.lst-kix_list_5-6{list-style-type:none}ul.lst-kix_t7qwyvm9772b-8{list-style-type:none}ul.lst-kix_t7qwyvm9772b-7{list-style-type:none}.lst-kix_list_4-0&gt;li{counter-increment:lst-ctn-kix_list_4-0}ul.lst-kix_t7qwyvm9772b-4{list-style-type:none}ul.lst-kix_list_5-0{list-style-type:none}ul.lst-kix_t7qwyvm9772b-3{list-style-type:none}.lst-kix_list_3-4&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_3-4,lower-latin) &quot;.
&quot;}ul.lst-kix_t7qwyvm9772b-6{list-style-type:none}ul.lst-kix_t7qwyvm9772b-5{list-style-type:none}ul.lst-kix_list_5-3{list-style-type:none}.lst-kix_list_3-3&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_3-3,decimal) &quot;.
&quot;}ol.lst-kix_list_3-5{list-style-type:none}ul.lst-kix_t7qwyvm9772b-0{list-style-type:none}ul.lst-kix_list_5-4{list-style-type:none}ol.lst-kix_list_3-6{list-style-type:none}ul.lst-kix_list_5-1{list-style-type:none}.lst-kix_list_8-0&gt;li:before{content:&quot;\0025cf
 &quot;}ol.lst-kix_list_3-7{list-style-type:none}ul.lst-kix_t7qwyvm9772b-2{list-style-type:none}.lst-kix_gzjq6lzg1wwb-3&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_gzjq6lzg1wwb-3,decimal) &quot;.
&quot;}ul.lst-kix_list_5-2{list-style-type:none}ol.lst-kix_list_3-8{list-style-type:none}ul.lst-kix_t7qwyvm9772b-1{list-style-type:none}.lst-kix_list_8-7&gt;li:before{content:&quot;\0025cb
 &quot;}.lst-kix_gzjq6lzg1wwb-4&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_gzjq6lzg1wwb-4,lower-latin) &quot;.
&quot;}.lst-kix_list_3-8&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_list_3-8,lower-roman) &quot;.
&quot;}.lst-kix_list_8-3&gt;li:before{content:&quot;\0025cf  &quot;}.lst-kix_gzjq6lzg1wwb-8&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_gzjq6lzg1wwb-8,lower-roman) &quot;.
&quot;}.lst-kix_oai74q1k79ui-8&gt;li{counter-increment:lst-ctn-kix_oai74q1k79ui-8}.lst-kix_list_3-7&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_3-7,lower-latin) &quot;. &quot;}.lst-kix_list_8-4&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_gzjq6lzg1wwb-7&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_gzjq6lzg1wwb-7,lower-latin) &quot;.
&quot;}ol.lst-kix_nua3ue1dzck6-8.start{counter-reset:lst-ctn-kix_nua3ue1dzck6-8
0}.lst-kix_list_8-8&gt;li:before{content:&quot;\0025a0  &quot;}.lst-kix_s49wx74p68ik-6&gt;li:before{content:&quot;\0025cf 
&quot;}.lst-kix_nua3ue1dzck6-0&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_nua3ue1dzck6-0,upper-latin) &quot;.
&quot;}.lst-kix_jphjql4uq7tq-6&gt;li:before{content:&quot;\0025cf 
&quot;}.lst-kix_jphjql4uq7tq-7&gt;li:before{content:&quot;\0025cb  &quot;}.lst-kix_nua3ue1dzck6-1&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_nua3ue1dzck6-1,lower-latin) &quot;. &quot;}.lst-kix_list_4-8&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_4-8,lower-roman) &quot;. &quot;}.lst-kix_nua3ue1dzck6-4&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_nua3ue1dzck6-4,lower-latin) &quot;. &quot;}.lst-kix_jphjql4uq7tq-2&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_jphjql4uq7tq-3&gt;li:before{content:&quot;\0025cf  &quot;}.lst-kix_list_4-7&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_4-7,lower-latin) &quot;) &quot;}.lst-kix_s49wx74p68ik-7&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_nua3ue1dzck6-8&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_nua3ue1dzck6-8,lower-roman) &quot;.
&quot;}.lst-kix_nua3ue1dzck6-5&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_nua3ue1dzck6-5,lower-roman) &quot;.
&quot;}ol.lst-kix_list_4-8.start{counter-reset:lst-ctn-kix_list_4-8 0}.lst-kix_s49wx74p68ik-2&gt;li:before{content:&quot;\0025a0 
&quot;}ol.lst-kix_list_3-3.start{counter-reset:lst-ctn-kix_list_3-3 0}.lst-kix_s49wx74p68ik-3&gt;li:before{content:&quot;\0025cf 
&quot;}.lst-kix_gzjq6lzg1wwb-0&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_gzjq6lzg1wwb-0,decimal) &quot;.
&quot;}.lst-kix_nua3ue1dzck6-2&gt;li{counter-increment:lst-ctn-kix_nua3ue1dzck6-2}.lst-kix_oai74q1k79ui-1&gt;li{counter-increment:lst-ctn-kix_oai74q1k79ui-1}.lst-kix_oai74q1k79ui-4&gt;li{counter-increment:lst-ctn-kix_oai74q1k79ui-4}.lst-kix_list_7-0&gt;li:before{content:&quot;\0025cf
 &quot;}ol.lst-kix_list_4-7.start{counter-reset:lst-ctn-kix_list_4-7
0}.lst-kix_list_3-7&gt;li{counter-increment:lst-ctn-kix_list_3-7}.lst-kix_gzjq6lzg1wwb-0&gt;li{counter-increment:lst-ctn-kix_gzjq6lzg1wwb-0}.lst-kix_list_2-4&gt;li:before{content:&quot;\0025a0
 &quot;}.lst-kix_list_2-8&gt;li:before{content:&quot;\0025c6 
&quot;}.lst-kix_nua3ue1dzck6-0&gt;li{counter-increment:lst-ctn-kix_nua3ue1dzck6-0}.lst-kix_oai74q1k79ui-4&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_oai74q1k79ui-4,lower-latin) &quot;. &quot;}.lst-kix_list_7-3&gt;li:before{content:&quot;\0025cf 
&quot;}.lst-kix_b1xgmn9bjew7-5&gt;li:before{content:&quot;\0025a0 
&quot;}ul.lst-kix_list_7-5{list-style-type:none}ul.lst-kix_list_7-6{list-style-type:none}ul.lst-kix_list_7-3{list-style-type:none}ul.lst-kix_list_7-4{list-style-type:none}ul.lst-kix_list_7-7{list-style-type:none}ul.lst-kix_list_7-8{list-style-type:none}ul.lst-kix_list_7-1{list-style-type:none}ul.lst-kix_list_7-2{list-style-type:none}ol.lst-kix_list_3-8.start{counter-reset:lst-ctn-kix_list_3-8
0}ol.lst-kix_nua3ue1dzck6-7.start{counter-reset:lst-ctn-kix_nua3ue1dzck6-7
0}.lst-kix_oai74q1k79ui-8&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_oai74q1k79ui-8,lower-roman) &quot;.
&quot;}ul.lst-kix_list_7-0{list-style-type:none}.lst-kix_list_7-7&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_b1xgmn9bjew7-1&gt;li:before{content:&quot;\0025cb 
&quot;}ol.lst-kix_nua3ue1dzck6-4.start{counter-reset:lst-ctn-kix_nua3ue1dzck6-4
0}.lst-kix_list_4-0&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_list_4-0,decimal) &quot;.
&quot;}.lst-kix_list_4-4&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_list_4-4,lower-latin) &quot;)
&quot;}ol.lst-kix_list_4-0{list-style-type:none}ol.lst-kix_list_4-1{list-style-type:none}ol.lst-kix_list_4-2{list-style-type:none}.lst-kix_oai74q1k79ui-6&gt;li{counter-increment:lst-ctn-kix_oai74q1k79ui-6}ol.lst-kix_list_4-3{list-style-type:none}ol.lst-kix_list_3-6.start{counter-reset:lst-ctn-kix_list_3-6
0}ol.lst-kix_nua3ue1dzck6-5.start{counter-reset:lst-ctn-kix_nua3ue1dzck6-5
0}ul.lst-kix_list_6-6{list-style-type:none}ul.lst-kix_list_6-7{list-style-type:none}ul.lst-kix_list_6-4{list-style-type:none}ul.lst-kix_list_6-5{list-style-type:none}ul.lst-kix_list_6-8{list-style-type:none}ol.lst-kix_list_4-8{list-style-type:none}.lst-kix_list_1-0&gt;li:before{content:&quot;\0025cf
 &quot;}ol.lst-kix_list_4-4{list-style-type:none}ul.lst-kix_list_6-2{list-style-type:none}ol.lst-kix_list_4-5{list-style-type:none}ul.lst-kix_list_6-3{list-style-type:none}ol.lst-kix_list_4-6{list-style-type:none}ul.lst-kix_list_6-0{list-style-type:none}ol.lst-kix_list_4-7{list-style-type:none}ul.lst-kix_list_6-1{list-style-type:none}.lst-kix_9hu4a59rs9m9-2&gt;li:before{content:&quot;\0025aa
 &quot;}.lst-kix_list_1-4&gt;li:before{content:&quot;\0025cb  &quot;}ol.lst-kix_list_3-5.start{counter-reset:lst-ctn-kix_list_3-5
0}.lst-kix_gzjq6lzg1wwb-7&gt;li{counter-increment:lst-ctn-kix_gzjq6lzg1wwb-7}.lst-kix_yefcnq5qqs1-2&gt;li:before{content:&quot;\0025a0
 &quot;}ol.lst-kix_nua3ue1dzck6-6.start{counter-reset:lst-ctn-kix_nua3ue1dzck6-6
0}.lst-kix_list_2-0&gt;li:before{content:&quot;\0025cf 
&quot;}.lst-kix_list_4-5&gt;li{counter-increment:lst-ctn-kix_list_4-5}.lst-kix_list_1-8&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_oai74q1k79ui-0&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_oai74q1k79ui-0,decimal) &quot;.
&quot;}.lst-kix_list_4-1&gt;li{counter-increment:lst-ctn-kix_list_4-1}.lst-kix_t7qwyvm9772b-0&gt;li:before{content:&quot;\0025cf 
&quot;}ul.lst-kix_list_1-0{list-style-type:none}.lst-kix_t7qwyvm9772b-2&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_gzjq6lzg1wwb-6&gt;li{counter-increment:lst-ctn-kix_gzjq6lzg1wwb-6}.lst-kix_t7qwyvm9772b-1&gt;li:before{content:&quot;\0025cb
 &quot;}.lst-kix_t7qwyvm9772b-3&gt;li:before{content:&quot;\0025cf 
&quot;}ol.lst-kix_list_3-1.start{counter-reset:lst-ctn-kix_list_3-1 0}.lst-kix_yefcnq5qqs1-7&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_yefcnq5qqs1-6&gt;li:before{content:&quot;\0025cf  &quot;}.lst-kix_yefcnq5qqs1-8&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_yefcnq5qqs1-5&gt;li:before{content:&quot;\0025a0 
&quot;}ul.lst-kix_list_1-3{list-style-type:none}ul.lst-kix_list_1-4{list-style-type:none}ul.lst-kix_list_1-1{list-style-type:none}ul.lst-kix_list_1-2{list-style-type:none}ul.lst-kix_list_1-7{list-style-type:none}ul.lst-kix_list_1-8{list-style-type:none}ul.lst-kix_list_1-5{list-style-type:none}ul.lst-kix_list_1-6{list-style-type:none}ol.lst-kix_list_4-5.start{counter-reset:lst-ctn-kix_list_4-5
0}ol.lst-kix_oai74q1k79ui-8.start{counter-reset:lst-ctn-kix_oai74q1k79ui-8
0}ol.lst-kix_oai74q1k79ui-3.start{counter-reset:lst-ctn-kix_oai74q1k79ui-3
0}.lst-kix_gzjq6lzg1wwb-2&gt;li{counter-increment:lst-ctn-kix_gzjq6lzg1wwb-2}ol.lst-kix_gzjq6lzg1wwb-8.start{counter-reset:lst-ctn-kix_gzjq6lzg1wwb-8
0}.lst-kix_list_3-0&gt;li{counter-increment:lst-ctn-kix_list_3-0}ol.lst-kix_list_4-0.start{counter-reset:lst-ctn-kix_list_4-0
0}.lst-kix_nua3ue1dzck6-5&gt;li{counter-increment:lst-ctn-kix_nua3ue1dzck6-5}.lst-kix_t7qwyvm9772b-6&gt;li:before{content:&quot;\0025cf
 &quot;}.lst-kix_t7qwyvm9772b-5&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_t7qwyvm9772b-7&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_t7qwyvm9772b-4&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_t7qwyvm9772b-8&gt;li:before{content:&quot;\0025a0 
&quot;}ol.lst-kix_gzjq6lzg1wwb-1.start{counter-reset:lst-ctn-kix_gzjq6lzg1wwb-1
0}.lst-kix_oai74q1k79ui-7&gt;li{counter-increment:lst-ctn-kix_oai74q1k79ui-7}.lst-kix_list_2-7&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_b1xgmn9bjew7-4&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_b1xgmn9bjew7-6&gt;li:before{content:&quot;\0025cf  &quot;}.lst-kix_list_2-5&gt;li:before{content:&quot;\0025c6 
&quot;}.lst-kix_oai74q1k79ui-3&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_oai74q1k79ui-3,decimal) &quot;.
&quot;}.lst-kix_oai74q1k79ui-3&gt;li{counter-increment:lst-ctn-kix_oai74q1k79ui-3}.lst-kix_oai74q1k79ui-5&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_oai74q1k79ui-5,lower-roman) &quot;.
&quot;}ol.lst-kix_gzjq6lzg1wwb-5{list-style-type:none}ol.lst-kix_gzjq6lzg1wwb-6{list-style-type:none}ol.lst-kix_gzjq6lzg1wwb-7{list-style-type:none}ol.lst-kix_gzjq6lzg1wwb-8{list-style-type:none}ol.lst-kix_gzjq6lzg1wwb-1{list-style-type:none}ol.lst-kix_gzjq6lzg1wwb-2{list-style-type:none}ol.lst-kix_gzjq6lzg1wwb-3{list-style-type:none}ol.lst-kix_gzjq6lzg1wwb-4{list-style-type:none}.lst-kix_b1xgmn9bjew7-0&gt;li:before{content:&quot;\0025cf
 &quot;}.lst-kix_b1xgmn9bjew7-2&gt;li:before{content:&quot;\0025a0 
&quot;}ol.lst-kix_gzjq6lzg1wwb-0{list-style-type:none}ol.lst-kix_list_4-3.start{counter-reset:lst-ctn-kix_list_4-3
0}.lst-kix_oai74q1k79ui-7&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_oai74q1k79ui-7,lower-latin) &quot;.
&quot;}ul.lst-kix_s49wx74p68ik-0{list-style-type:none}ul.lst-kix_s49wx74p68ik-1{list-style-type:none}ul.lst-kix_s49wx74p68ik-2{list-style-type:none}ol.lst-kix_nua3ue1dzck6-2.start{counter-reset:lst-ctn-kix_nua3ue1dzck6-2
0}.lst-kix_list_3-4&gt;li{counter-increment:lst-ctn-kix_list_3-4}ul.lst-kix_list_2-8{list-style-type:none}ul.lst-kix_list_2-2{list-style-type:none}ul.lst-kix_list_2-3{list-style-type:none}ul.lst-kix_list_2-0{list-style-type:none}ul.lst-kix_list_2-1{list-style-type:none}ul.lst-kix_list_2-6{list-style-type:none}.lst-kix_list_1-1&gt;li:before{content:&quot;\0025cb
 &quot;}ul.lst-kix_list_2-7{list-style-type:none}ul.lst-kix_list_2-4{list-style-type:none}ul.lst-kix_list_2-5{list-style-type:none}ul.lst-kix_s49wx74p68ik-3{list-style-type:none}.lst-kix_list_1-3&gt;li:before{content:&quot;\0025cf
 &quot;}ul.lst-kix_s49wx74p68ik-4{list-style-type:none}ul.lst-kix_s49wx74p68ik-5{list-style-type:none}ul.lst-kix_s49wx74p68ik-6{list-style-type:none}ul.lst-kix_s49wx74p68ik-7{list-style-type:none}.lst-kix_yefcnq5qqs1-1&gt;li:before{content:&quot;\0025cb
 &quot;}ul.lst-kix_s49wx74p68ik-8{list-style-type:none}.lst-kix_yefcnq5qqs1-3&gt;li:before{content:&quot;\0025cf 
&quot;}.lst-kix_list_4-8&gt;li{counter-increment:lst-ctn-kix_list_4-8}.lst-kix_list_1-7&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_list_1-5&gt;li:before{content:&quot;\0025a0 
&quot;}ol.lst-kix_oai74q1k79ui-6.start{counter-reset:lst-ctn-kix_oai74q1k79ui-6
0}ol.lst-kix_oai74q1k79ui-0{list-style-type:none}ol.lst-kix_nua3ue1dzck6-0.start{counter-reset:lst-ctn-kix_nua3ue1dzck6-0
0}ol.lst-kix_oai74q1k79ui-1{list-style-type:none}.lst-kix_list_2-1&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_oai74q1k79ui-1&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_oai74q1k79ui-1,lower-latin) &quot;.
&quot;}ol.lst-kix_oai74q1k79ui-8{list-style-type:none}.lst-kix_list_2-3&gt;li:before{content:&quot;\0025cf 
&quot;}ol.lst-kix_oai74q1k79ui-6{list-style-type:none}.lst-kix_b1xgmn9bjew7-8&gt;li:before{content:&quot;\0025a0 
&quot;}ol.lst-kix_oai74q1k79ui-7{list-style-type:none}ol.lst-kix_oai74q1k79ui-4{list-style-type:none}ol.lst-kix_oai74q1k79ui-5{list-style-type:none}ol.lst-kix_oai74q1k79ui-2{list-style-type:none}ol.lst-kix_oai74q1k79ui-3{list-style-type:none}.lst-kix_list_3-1&gt;li{counter-increment:lst-ctn-kix_list_3-1}.lst-kix_list_3-1&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_3-1,lower-latin) &quot;. &quot;}.lst-kix_list_3-2&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_3-2,lower-roman) &quot;. &quot;}.lst-kix_list_8-1&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_list_8-2&gt;li:before{content:&quot;\0025a0  &quot;}.lst-kix_list_3-5&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_3-5,lower-roman) &quot;. &quot;}.lst-kix_gzjq6lzg1wwb-2&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_gzjq6lzg1wwb-2,lower-roman) &quot;.
&quot;}.lst-kix_nua3ue1dzck6-6&gt;li{counter-increment:lst-ctn-kix_nua3ue1dzck6-6}.lst-kix_list_8-5&gt;li:before{content:&quot;\0025a0
 &quot;}ol.lst-kix_gzjq6lzg1wwb-5.start{counter-reset:lst-ctn-kix_gzjq6lzg1wwb-5
0}.lst-kix_list_8-6&gt;li:before{content:&quot;\0025cf  &quot;}.lst-kix_gzjq6lzg1wwb-5&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_gzjq6lzg1wwb-5,lower-roman) &quot;. &quot;}.lst-kix_list_3-6&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_3-6,decimal) &quot;. &quot;}.lst-kix_gzjq6lzg1wwb-6&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_gzjq6lzg1wwb-6,decimal) &quot;.
&quot;}.lst-kix_nua3ue1dzck6-4&gt;li{counter-increment:lst-ctn-kix_nua3ue1dzck6-4}ol.lst-kix_oai74q1k79ui-5.start{counter-reset:lst-ctn-kix_oai74q1k79ui-5
0}ol.lst-kix_list_4-2.start{counter-reset:lst-ctn-kix_list_4-2 0}.lst-kix_jphjql4uq7tq-8&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_s49wx74p68ik-5&gt;li:before{content:&quot;\0025a0  &quot;}.lst-kix_nua3ue1dzck6-2&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_nua3ue1dzck6-2,lower-roman) &quot;. &quot;}.lst-kix_jphjql4uq7tq-5&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_gzjq6lzg1wwb-3&gt;li{counter-increment:lst-ctn-kix_gzjq6lzg1wwb-3}.lst-kix_list_4-4&gt;li{counter-increment:lst-ctn-kix_list_4-4}.lst-kix_nua3ue1dzck6-3&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_nua3ue1dzck6-3,decimal) &quot;. &quot;}.lst-kix_s49wx74p68ik-1&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_s49wx74p68ik-0&gt;li:before{content:&quot;\0025cf 
&quot;}.lst-kix_s49wx74p68ik-8&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_jphjql4uq7tq-4&gt;li:before{content:&quot;\0025cb  &quot;}.lst-kix_nua3ue1dzck6-7&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_nua3ue1dzck6-7,lower-latin) &quot;. &quot;}ol.lst-kix_list_4-1.start{counter-reset:lst-ctn-kix_list_4-1
0}.lst-kix_nua3ue1dzck6-6&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_nua3ue1dzck6-6,decimal) &quot;.
&quot;}.lst-kix_jphjql4uq7tq-1&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_s49wx74p68ik-4&gt;li:before{content:&quot;\0025cb 
&quot;}.lst-kix_jphjql4uq7tq-0&gt;li:before{content:&quot;\0025cf 
&quot;}ol.lst-kix_gzjq6lzg1wwb-0.start{counter-reset:lst-ctn-kix_gzjq6lzg1wwb-0
0}.lst-kix_gzjq6lzg1wwb-1&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_gzjq6lzg1wwb-1,lower-latin) &quot;.
&quot;}.lst-kix_nua3ue1dzck6-8&gt;li{counter-increment:lst-ctn-kix_nua3ue1dzck6-8}.lst-kix_list_3-3&gt;li{counter-increment:lst-ctn-kix_list_3-3}.lst-kix_list_2-6&gt;li:before{content:&quot;\0025cf
 &quot;}.lst-kix_list_7-1&gt;li:before{content:&quot;\0025a0  &quot;}.lst-kix_list_7-5&gt;li:before{content:&quot;\0025c6 
&quot;}.lst-kix_b1xgmn9bjew7-3&gt;li:before{content:&quot;\0025cf 
&quot;}.lst-kix_b1xgmn9bjew7-7&gt;li:before{content:&quot;\0025cb 
&quot;}ol.lst-kix_nua3ue1dzck6-8{list-style-type:none}ol.lst-kix_nua3ue1dzck6-7{list-style-type:none}ol.lst-kix_gzjq6lzg1wwb-6.start{counter-reset:lst-ctn-kix_gzjq6lzg1wwb-6
0}ol.lst-kix_nua3ue1dzck6-0{list-style-type:none}ol.lst-kix_nua3ue1dzck6-2{list-style-type:none}ol.lst-kix_nua3ue1dzck6-1{list-style-type:none}ol.lst-kix_nua3ue1dzck6-4{list-style-type:none}ol.lst-kix_oai74q1k79ui-1.start{counter-reset:lst-ctn-kix_oai74q1k79ui-1
0}ol.lst-kix_nua3ue1dzck6-3{list-style-type:none}.lst-kix_oai74q1k79ui-6&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_oai74q1k79ui-6,decimal) &quot;.
&quot;}ol.lst-kix_nua3ue1dzck6-6{list-style-type:none}ol.lst-kix_nua3ue1dzck6-5{list-style-type:none}ol.lst-kix_gzjq6lzg1wwb-3.start{counter-reset:lst-ctn-kix_gzjq6lzg1wwb-3
0}.lst-kix_oai74q1k79ui-5&gt;li{counter-increment:lst-ctn-kix_oai74q1k79ui-5}ul.lst-kix_yefcnq5qqs1-5{list-style-type:none}.lst-kix_list_3-8&gt;li{counter-increment:lst-ctn-kix_list_3-8}ul.lst-kix_yefcnq5qqs1-4{list-style-type:none}ul.lst-kix_yefcnq5qqs1-3{list-style-type:none}ul.lst-kix_yefcnq5qqs1-2{list-style-type:none}ul.lst-kix_yefcnq5qqs1-8{list-style-type:none}.lst-kix_list_4-6&gt;li{counter-increment:lst-ctn-kix_list_4-6}ul.lst-kix_yefcnq5qqs1-7{list-style-type:none}ul.lst-kix_yefcnq5qqs1-6{list-style-type:none}.lst-kix_list_4-2&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_4-2,lower-roman) &quot;. &quot;}.lst-kix_list_4-6&gt;li:before{content:&quot;&quot;
counter(lst-ctn-kix_list_4-6,decimal) &quot;.
&quot;}ul.lst-kix_yefcnq5qqs1-1{list-style-type:none}ul.lst-kix_yefcnq5qqs1-0{list-style-type:none}.lst-kix_nua3ue1dzck6-1&gt;li{counter-increment:lst-ctn-kix_nua3ue1dzck6-1}ol.lst-kix_gzjq6lzg1wwb-4.start{counter-reset:lst-ctn-kix_gzjq6lzg1wwb-4
0}.lst-kix_oai74q1k79ui-0&gt;li{counter-increment:lst-ctn-kix_oai74q1k79ui-0}.lst-kix_list_1-2&gt;li:before{content:&quot;\0025a0 
&quot;}.lst-kix_yefcnq5qqs1-0&gt;li:before{content:&quot;\0025cf  &quot;}.lst-kix_9hu4a59rs9m9-0&gt;li:before{content:&quot;\0025cf
 &quot;}.lst-kix_yefcnq5qqs1-4&gt;li:before{content:&quot;\0025cb  &quot;}.lst-kix_list_1-6&gt;li:before{content:&quot;\0025cf 
&quot;}.lst-kix_gzjq6lzg1wwb-1&gt;li{counter-increment:lst-ctn-kix_gzjq6lzg1wwb-1}ol.lst-kix_oai74q1k79ui-0.start{counter-reset:lst-ctn-kix_oai74q1k79ui-0
0}.lst-kix_oai74q1k79ui-2&gt;li:before{content:&quot;&quot; counter(lst-ctn-kix_oai74q1k79ui-2,lower-roman) &quot;.
&quot;}.lst-kix_list_2-2&gt;li:before{content:&quot;\0025c6  &quot;}ol{margin:0;padding:0}table td,table
th{padding:0}.c70{border-right-style:solid;padding:5pt 5pt 5pt
5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#f3f3f3;border-left-style:solid;border-bottom-width:1pt;width:468pt;border-top-color:#000000;border-bottom-style:solid}.c38{border-right-style:solid;padding:0pt
5.8pt 0pt
5.8pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:72pt;border-top-color:#000000;border-bottom-style:solid}.c41{border-right-style:solid;padding:0pt
5.8pt 0pt
5.8pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:144pt;border-top-color:#000000;border-bottom-style:solid}.c7{border-right-style:solid;padding:0pt
5.8pt 0pt
5.8pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:220.5pt;border-top-color:#000000;border-bottom-style:solid}.c54{border-right-style:solid;padding:0pt
5.8pt 0pt
5.8pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:139.5pt;border-top-color:#000000;border-bottom-style:solid}.c48{border-right-style:solid;padding:0pt
5.8pt 0pt
5.8pt;border-bottom-color:#000000;border-top-width:0pt;border-right-width:0pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;width:211.5pt;border-top-color:#000000;border-bottom-style:solid}.c27{border-right-style:solid;padding:0pt
5.8pt 0pt
5.8pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:middle;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:138pt;border-top-color:#000000;border-bottom-style:solid}.c0{border-right-style:solid;padding:0pt
5.8pt 0pt
5.8pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:middle;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:159.8pt;border-top-color:#000000;border-bottom-style:solid}.c23{border-right-style:solid;padding:0pt
5.8pt 0pt
5.8pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:middle;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:152.2pt;border-top-color:#000000;border-bottom-style:solid}.c6{color:#000000;font-weight:normal;text-decoration:none;vertical-align:baseline;font-size:7pt;font-family:&quot;Courier
New&quot;;font-style:normal}.c14{color:#000090;font-weight:bold;text-decoration:none;vertical-align:baseline;font-size:18pt;font-family:&quot;Cambria&quot;;font-style:normal}.c1{color:#000000;font-weight:normal;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:&quot;Calibri&quot;;font-style:normal}.c37{color:#000000;font-weight:normal;text-decoration:none;vertical-align:baseline;font-family:&quot;Arial&quot;;font-style:normal}.c31{margin-left:16.5pt;padding-top:6pt;padding-left:2.2pt;padding-bottom:9pt;page-break-after:avoid}.c30{margin-left:36pt;padding-top:0pt;padding-bottom:10pt;line-height:1.0;text-align:justify}.c3{padding-top:5pt;padding-bottom:5pt;orphans:2;text-indent:36pt;widows:2}.c68{margin-left:11.9pt;border-spacing:0;border-collapse:collapse;margin-right:auto}.c77{margin-left:16.8pt;border-spacing:0;border-collapse:collapse;margin-right:auto}.c57{border-spacing:0;border-collapse:collapse;margin-right:auto}.c20{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c10{padding-top:0pt;padding-bottom:0pt;line-height:1.15;text-align:left}.c15{padding-top:5pt;padding-bottom:5pt;text-align:center}.c12{padding-top:0pt;padding-bottom:5pt;height:11pt}.c4{font-size:18pt;font-weight:bold;text-decoration:underline}.c88{background-color:#ffffff;max-width:468pt;padding:72pt
72pt 72pt
72pt}.c71{padding-top:6pt;padding-bottom:9pt;height:18pt}.c36{padding-top:5pt;padding-bottom:0pt;line-height:1.0}.c13{padding-top:5pt;padding-bottom:0pt;text-indent:36pt}.c29{margin-left:36pt;padding-left:0pt}.c75{font-size:7pt;font-family:&quot;Courier
New&quot;}.c11{color:inherit;text-decoration:inherit}.c67{font-size:8pt;font-family:&quot;Courier
New&quot;}.c16{padding-top:5pt;padding-bottom:5pt}.c9{orphans:2;widows:2}.c56{padding:0;margin:0}.c40{border:1px solid
black;margin:5px}.c58{padding-top:12pt;padding-bottom:8pt}.c43{padding-top:5pt;padding-bottom:0pt}.c33{color:#1155cc;text-decoration:underline}.c18{padding-top:12pt;page-break-after:avoid}.c2{background-color:#ffffff;color:#333333}.c24{font-family:&quot;Cambria&quot;;font-weight:bold}.c60{padding-top:6pt;height:14pt}.c39{padding-top:0pt;padding-bottom:0pt}.c46{background-color:#ffffff;color:#2e2e2e}.c8{font-family:&quot;Calibri&quot;;color:#7c4705}.c19{color:#000090;font-size:13pt}.c80{margin-left:40.5pt;padding-left:-4.5pt}.c5{font-size:11pt}.c51{color:#1155cc}.c72{font-size:18pt}.c63{margin-left:18pt}.c62{font-style:italic}.c84{font-family:&quot;Arial&quot;}.c82{height:12pt}.c74{padding-top:6pt}.c66{text-align:right}.c83{padding-bottom:2pt}.c73{font-weight:normal}.c79{padding-bottom:9pt}.c64{text-decoration:none}.c86{padding-bottom:12pt}.c53{page-break-after:avoid}.c42{padding-top:0pt}.c52{margin-left:36pt}.c49{font-size:12pt}.c85{color:#632523}.c45{line-height:1.0}.c65{font-size:10pt}.c44{height:13pt}.c34{padding-bottom:5pt}.c78{padding-bottom:0pt}.c59{text-align:justify}.c76{margin-left:72pt}.c50{text-decoration:underline}.c35{font-weight:bold}.c17{height:0pt}.c25{color:#3376be}.c69{padding-left:0pt}.c26{font-family:&quot;Calibri&quot;}.c55{text-align:center}.c87{color:#000000}.c61{padding-bottom:10pt}.c47{color:#0000ff}.c28{color:#000090}.c21{text-align:left}.c81{padding-bottom:36pt}.c32{color:#7c4705}.c22{height:11pt}.title{padding-top:24pt;color:#000000;font-weight:bold;font-size:36pt;padding-bottom:6pt;font-family:&quot;Cambria&quot;;line-height:1.0;page-break-after:avoid;text-align:justify}.subtitle{padding-top:18pt;color:#666666;font-size:24pt;padding-bottom:4pt;font-family:&quot;Georgia&quot;;line-height:1.0;page-break-after:avoid;font-style:italic;text-align:justify}li{color:#000000;font-size:11pt;font-family:&quot;Calibri&quot;}p{margin:0;color:#000000;font-size:11pt;font-family:&quot;Calibri&quot;}h1{padding-top:24pt;color:#000000;font-weight:bold;font-size:18pt;padding-bottom:6pt;font-family:&quot;Cambria&quot;;line-height:1.0;page-break-after:avoid;text-align:left}h2{padding-top:5pt;color:#000090;font-weight:bold;font-size:13pt;padding-bottom:5pt;font-family:&quot;Calibri&quot;;line-height:1.15;page-break-after:avoid;text-align:left}h3{padding-top:14pt;color:#000000;font-weight:bold;font-size:14pt;padding-bottom:4pt;font-family:&quot;Cambria&quot;;line-height:1.0;page-break-after:avoid;text-align:justify}h4{padding-top:12pt;color:#000000;font-weight:bold;font-size:12pt;padding-bottom:2pt;font-family:&quot;Cambria&quot;;line-height:1.0;page-break-after:avoid;text-align:justify}h5{padding-top:11pt;color:#000000;font-weight:bold;font-size:11pt;padding-bottom:2pt;font-family:&quot;Cambria&quot;;line-height:1.0;page-break-after:avoid;text-align:justify}h6{padding-top:10pt;color:#000000;font-weight:bold;font-size:10pt;padding-bottom:2pt;font-family:&quot;Cambria&quot;;line-height:1.0;page-break-after:avoid;text-align:justify}
</style>
    <title></title>
  </head>
  <body class="c88">
    <div>
      <p class="c42 c45 c59 c61 c22"></p>
    </div>
    <p class="c9 c21">
      <span class="c72 c35">Implementation Guide Draft Version 1</span>
    </p>
    <p class="c15 c9">
      <span class="c35 c72"></span>
    </p>
    <p class="c15 c9">
      <span class="c4">Table of Contents</span>
    </p>
    <p class="c22"></p>
    <ol class="c56 lst-kix_list_3-0 start" start="1">
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.gjdgxs">Background</a>
        </span>
      </li>
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.3znysh7">Overview</a>
        </span>
      </li>
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.4i7ojhp">Sequence Resource</a>
        </span>
      </li>
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.49x2ik5">Observation-genetics Profile</a>
        </span>
      </li>
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.32hioqz">DiagnosticReport-genetics Profile</a>
        </span>
      </li>
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.4f1mdlm">DiagnosticOrder-genetics Profile</a>
        </span>
      </li>
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.nmf14n">HLA genotyping results Profile</a>
        </span>
      </li>
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.tf76y3w5j4lx">Relationship among Sequence resource and genetics profiles</a>
        </span>
      </li>
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.76drlbxj2vl8">HL7 Domain Analysis Model Use Cases</a>
        </span>
      </li>
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.vuedwvest6c9">Other Genomics Use Cases</a>
        </span>
      </li>
      <li class="c29 c39">
        <span class="c33">
          <a class="c11" href="#h.hb7rvdbqv2xg">Clinico-Genomic Apps</a>
        </span>
      </li>
    </ol>
    <p class="c39 c22"></p>
    <p class="c63">
      <span class="c33">
        <a class="c11" href="#h.pkwqa1">Appendix 1: From DSTU2 Genomics to STU3</a>
      </span>
    </p>
    <p class="c63 c22"></p>
    <p class="c16 c9 c22 c66"></p>
    <p class="c22"></p>
    <p class="c22">
      <span>
        <a class="c11" href="#h.uz79z378q86g"></a>
      </span>
    </p>
    <hr style="page-break-before:always;display:none;" />
    <p class="c22">
      <span>
        <a class="c11" href="#h.uz79z378q86g"></a>
      </span>
    </p>
    <p class="c22">
      <span>
        <a class="c11" href="#h.uz79z378q86g"></a>
      </span>
    </p>
    <ol class="c56 lst-kix_list_4-0 start" start="1">
      <li class="c31 c9">
        <h1 id="h.gjdgxs" style="display:inline">
          <span class="c28">Background</span>
        </h1>
      </li>
    </ol>
    <p class="c16 c9 c45 c59" id="h.pms6ytlr44fw">
    <span class="c5">R</span>
    <span class="c5">apid advances in sequencing technologies, clinical genetics tests for whole genome and exome sequencing
    are</span> 
    <span>allowing</span>
    <span class="c5">sophisticated genetics testing</span> 
    <span>to be used by providers and patients in clinical decisions</span>
    <span class="c5">. Results from such tests</span> 
    <span>are</span>
    <span class="c5">used to identify distinct genetic variants</span> 
    <span>that</span>
    <span class="c5"></span>
    <span>may contribute to syndromes,</span> 
    <span class="c5">conditions and/or predictive respons</span>
    <span>es to</span> 
    <span class="c5">treatments. The</span> 
    <span>implementation</span>
    <span class="c5">of precision medicine will depend upon</span> 
    <span>having</span>
    <span class="c5">such data to diagnose patients, choose medications, and predict the course of disease and care, b</span>
    <span>ut will require standards and effective user interfaces</span>
    <span class="c5">.</span></p>
    <p class="c16 c9 c45 c59" id="h.sl59u8yrsw99">
    <span>A</span>
    <span class="c5">current technical challenge</span> 
    <span>exists in interoperability, or allowing the ability to access and</span>
    <span class="c5">share clinical and genetics data</span>
    <span>. This interoperability</span>
    <span class="c5">includ</span>
    <span>es</span>
    <span class="c5">collection, coding, and retrieval</span> 
    <span>to</span>
    <span class="c5">scale.</span> 
    <span>An individual's</span> 
    <span class="c5">genetic data set</span> 
    <span>is</span>
    <span class="c5">large</span>
    <span>,</span> 
    <span class="c5">complex and requires curation.</span> 
    <span>Unfortunately,</span> 
    <span class="c5">incompatible systems and nomenclatures are already in use. A standards based ontology that could be adopted to
    integrate both</span> 
    <span>genetic data</span>
    <span class="c5"></span>
    <span class="c5 c62">and</span>
    <span class="c5">clinical information systems will be crucial to</span>
    <span></span>
    <span class="c5">accelerating the integration of precision medicine and to make sense of genetic testing results in a complete
    clinical context.</span></p>
    <p class="c16 c9 c45 c59" id="h.mg3hm9l785jq">
      <span class="c5">One approach for c</span>
      <span>ollecting</span>
      <span class="c5">, coding, and retrieving genetics data comes from the Global Alliance for Genomics and Health (</span>
      <span class="c5 c35">GA4GH</span>
      <span class="c5">; http://genomicsandhealth.org/). The GA4GH organization has built and is refining an API and data model for
      the exchange of full sequence genomic information across multiple research organizations and platforms. The GA4GH focuses on
      the needs of researchers.</span>
    </p>
    <p class="c16 c9 c45 c59" id="h.f3nl5j4rwtfs">
    <span class="c5">A second approach is</span> 
    <span>evolving</span>
    <span class="c5">from Health Level 7 (</span>
    <span class="c5 c35">HL7</span>
    <span class="c5">; hl7.org) through its standards</span> 
    <span>family</span> 
    <sup>
      <a href="#cmnt1" id="cmnt_ref1">[a]</a>
    </sup>
    <span class="c5">called Fast Healthcare Interoperability Resources (</span>
    <span class="c5 c35">FHIR</span>
    <span class="c5">; hl7.org/fhir). FHIR focuses on clinical needs and systems and has principally focused on standardizing
    traditional clinical data of the sort in EMR systems using very modern web technology-based ideas. FHIR is attractive because
    it is relatively easy to implement</span>
    <span>because</span>
    <span class="c5">it is comprised of a set of modular components called</span> 
    <span class="c5 c62">resources</span>
    <span class="c5">, which can be easily and incrementally assembled into working systems. The clinical requirements for genetics
    data is, relative to genomics research needs, utilitarian and reductive</span>
    <span>because</span> 
    <span class="c5">it is about distilling and extracting particular genetics data produced by ever more sophisticated testing for
    use at the point-of-care. This has made FHIR a</span>
    <span>very functional</span>
    <span class="c5">framework to</span> 
    <span>initiate an interoperable</span>
    <span class="c5">clinical genetics data standardization to which multiple stakeholders have contributed to this
    guide.</span></p>
    <p class="c16 c9 c45 c59" id="h.dljluw33klhm">
    <span class="c5">Starting in 2014, the HL7 Clinical Genomics Working Group (</span>
    <span class="c5 c35">CGWG</span>
    <span class="c5">) has been</span> 
    <span>drafting</span>
    <span class="c5">the FHIR framework for clinical genetics testing for general adoption. This effort reached its first major
    milestone on September 23, 2015, when HL7 released its second</span>
    <span></span>
    <span class="c5">FHIR Draft Standard for Trial Use (</span>
    <span class="c5 c35">DSTU2</span>
    <span class="c5">). DSTU2 introduced a</span> 
    <span class="c5 c51">
      <a class="c11"
      href="https://www.google.com/url?q=http://hl7.org/fhir/observation-genetics-cg-prf-1a.html&amp;sa=D&amp;ust=1468821991830000&amp;usg=AFQjCNFTpZihlbW6ciMC4c6PN8Fl1nvkxQ">
      Standard genetics profile</a>
    </span>
    <span class="c5">that applies to the FHIR</span> 
    <span class="c5 c51">
      <a class="c11"
      href="https://www.google.com/url?q=http://hl7.org/fhir/observation.html&amp;sa=D&amp;ust=1468821991831000&amp;usg=AFQjCNHNxaKl2TCp8PuesX_DU-sKmSd-5A">
      Observation resource</a>
    </span>
    <span class="c5">. The profile was created according to the FHIR Clinical Genomics Project Scope Statement that authorized its
    community development. Using this profile, Observation payloads can return genetic testing results in a standardized
    manner.</span></p>
    <p class="c16 c9 c45 c59" id="h.h1jquhfo20wg">
    <span class="c5">Extending Observations rather than creating a new, dedicated FHIR genetics data resource</span> 
    <span>is consistent</span>
    <span class="c5">with the FHIR community</span> 
    <span>mandate because</span> 
    <span class="c5">it only adds new resources to</span> 
    <span>an existing</span>
    <span class="c5">resource library when there is a compelling case to do so. We provide</span> 
    <span>the</span>
    <span class="c5">case to add a</span> 
    <span class="c5 c35">Sequence</span>
    <span class="c5">res</span>
    <span>ource</span> 
    <span class="c5">in this document.</span></p>
    <p class="c16 c9 c45 c59 c22" id="h.gdiezperp864"></p>
    <p class="c22"></p>
    <hr style="page-break-before:always;display:none;" />
    <p class="c22"></p>
    <p class="c22"></p>
    <ol class="c56 lst-kix_list_4-0" start="2">
      <li class="c31 c9">
        <h1 id="h.3znysh7" style="display:inline">
          <span class="c28">Overview</span>
        </h1>
      </li>
    </ol>
    <p class="c9" id="h.y4i0n1eaukls">
    <span class="c5">Given the clinical significance of gen</span>
    <span>etic</span>
    <span class="c5">sequence data and the wealth of</span> 
    <span>information that may be retrieved for an individual's health and with</span>
    <span class="c5">metadata</span> 
    <span>for the betterment of public health</span>
    <span class="c5">, we propose to move beyond FHIR DSTU2 Standard Genetics profile on Observation and allow increa</span>
    <span>s</span>
    <span class="c5">ed granularity and less ambiguity by creat</span>
    <span>ing</span>
    <span class="c5">a new resource to be called</span> 
    <span class="c5 c35">Sequence</span>
    <span class="c5">. This resource will be used to</span> 
    <span class="c5">h</span>
    <span>old</span> 
    <sup>
      <a href="#cmnt2" id="cmnt_ref2">[b]</a>
    </sup>
    <span class="c5">clinically relevant sequenc</span>
    <span>e</span>
    <span class="c5">data</span>
    <span class="c5">in a manner that is both efficient and versatile</span> 
    <span>integrating</span>
    <span class="c5">new and as yet</span> 
    <span class="c5">undefined types of genomic and other -omics data</span>
    <sup>
      <a href="#cmnt3" id="cmnt_ref3">[c]</a>
    </sup>
    <span class="c5">that will soon be commonly enter</span>
    <span>ed into</span>
    <span class="c5">health records for clinical use. </span>
    <span>Sequence will be leveraged by other FHIR resources, including Observation.</span> 
    <span>This is consistent</span>
    <span class="c5">with how all FHIR resources are</span> 
    <span>designed and used</span>
    <sup>
      <a href="#cmnt4" id="cmnt_ref4">[d]</a>
    </sup>
    <span>.</span></p>
    <p class="c16 c9">
    <span class="c5">The FHIR Clinical Genomics Working Group (CGWG) began discussing such a dedicated resource in 2014. The
    September 2014 Informative Ballot (</span>
    <span class="c5 c35">HL7</span> 
    <span class="c35">Clinical Genomics,</span> 
    <span class="c5 c35">Domain Analysis Model: Clinical Sequencing Release 1</span>
    <sup>
      <a href="#cmnt5" id="cmnt_ref5">[e]</a>
    </sup>
    <span class="c5 c35"></span>
    <span class="c5">) provided guiding use cases, which initially informed development of the initial Standard Genetics profile
    that is found in FHIR DSTU2. The same use cases also led to a second Project Scope Statement in May 2015 (approved by HL7
    DESD/OO/FMG) to develop a Sequence resource (</span>
    <span class="c5 c35">Develop FHIR sequence resource for Clinical Genomics</span>
    <span class="c5">). A pre</span>
    <span>liminary</span>
    <span class="c5">effort to address these issues ha</span>
    <span>s</span>
    <span class="c5">been explored and published</span> 
    <span>in context of the</span> 
    <span class="c2">Substitutable Medical Applications &amp; Reusable Technologies (</span>
    <span>SMART) Platforms Project</span> 
    <span class="c5">(</span>
    <span class="c33 c5">
      <a class="c11"
      href="https://www.google.com/url?q=http://smarthealthit.org/an-app-platform-for-healthcare/about/&amp;sa=D&amp;ust=1468821991847000&amp;usg=AFQjCNFmF15vm9f8weTsp2gxFuCYm3rD0Q">http://smarthealthit.org/an-app-platform-for-healthcare/about/</a>
    </span>
    <span class="c5">) and described in an article (</span>
    <span class="c5 c51">
      <a class="c11"
      href="https://www.google.com/url?q=http://jamia.oxfordjournals.org/content/early/2015/07/21/jamia.ocv045.long&amp;sa=D&amp;ust=1468821991848000&amp;usg=AFQjCNERd6q3E5wNxTCP_cOnK7r8DFmRFQ">SMART
      on FHIR Genomics: Facilitating standardized clinico-genomic apps</a>
    </span>
    <span class="c5">).</span></p>
    <p class="c16 c9" id="h.t5xtdyggd9fo">
    <span class="c5">Sequence is designed to</span> 
    <span>hold</span> 
    <sup>
      <a href="#cmnt6" id="cmnt_ref6">[f]</a>
    </sup>
    <span class="c5">gen</span>
    <span>etic</span>
    <span class="c5"></span>
    <span>sequences in blocks</span>
    <span class="c5">relevant to actionable clinical decision-making. Extensions to Sequence address complex cases, such as Human
    Leukocyte Antigen (HLA) genotyping, and can associate it</span>
    <span class="c5">with repositories for retrieving a patient's full sequence data, such as those defined by the Global Alliance
    for Genomics Health (</span>
    <span class="c5 c51">
      <a class="c11"
      href="https://www.google.com/url?q=http://genomicsandhealth.org&amp;sa=D&amp;ust=1468821991852000&amp;usg=AFQjCNFJk_GmgZKnK4tsnpbIIwSCAip5FA">GA4GH</a>
    </span>
    <span class="c5">). Other changes include a suite of genetics profiles for other FHIR resources. In addition, the
    Observation-genetics profile adds new references so that an Observation can report genetics test results</span>
    <span>to be integrated into the EHR</span>
    <span class="c5">. There are also new genetics-extension profiles for DiagnosticReport, DiagnosticOrder and
    FamilyMemberHistory, respectively, to extend them to report genetics results. We have given all of these FHIR genetics profiles
    the suffix -genetics (e.g. DiagnosticReport-genetics profile). New profiles on top of Observation and DiagnosticReport have
    been created for reporting HLA genotyping results.</span></p>
    <p class="c16 c9" id="h.tyjcwt">
    <span class="c5">On the following pages</span>
    <span class="c5">, we elaborate upon the rationale for the proposed design, introducing in some detail the following</span> 
    <span>resource and profiles</span>
    <sup>
      <a href="#cmnt7" id="cmnt_ref7">[g]</a>
    </sup>
    <span class="c5">:</span></p>
    <ul class="c56 lst-kix_list_8-0 start">
      <li class="c29 c9 c39" id="h.3dy6vkm">
        <span class="c5">A Sequence resource</span>
      </li>
      <li class="c29 c9 c39">
        <span class="c5">An Observation-genetics profile on Observation</span>
      </li>
      <li class="c29 c9 c39" id="h.1t3h5sf">
        <span class="c5">A DiagnosticReport-genetics profile on DiagnosticReport</span>
      </li>
      <li class="c29 c9 c39" id="h.4d34og8">
        <span class="c5">A FamilyMemberHistory-genetics profile on FamilyMemberHistory</span>
      </li>
      <li class="c29 c9 c39" id="h.2s8eyo1">
        <span class="c5">A DiagnosticOrder-genetics profile on DiagnosticOrder</span>
      </li>
      <li class="c29 c9 c39" id="h.17dp8vu">
        <span class="c5">An HLA-genotyping-results profile on DiagnosticReport</span>
      </li>
      <li class="c29 c9 c39" id="h.3rdcrjn">
      <span class="c5">A Consensus-sequence-block profile on</span> 
      <span>Sequence</span></li>
    </ul>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9 c45 c59">
    <span class="c5">With these</span> 
    <span>resource and profiles</span>
    <sup>
      <a href="#cmnt8" id="cmnt_ref8">[h]</a>
    </sup>
    <span class="c5">, FHIR can support a large set of clinical use cases</span>
    <span class="c5 c26">(see</span> 
    <span class="c5 c51 c26">
      <a class="c11" href="#h.76drlbxj2vl8">Section 9</a>
    </span>
    <span class="c5 c26">,</span> 
    <span class="c33 c5 c26">
      <a class="c11" href="#h.vuedwvest6c9">10</a>
    </span>
    <span class="c5 c26">, and</span> 
    <span class="c33 c5 c26">
      <a class="c11" href="#h.hb7rvdbqv2xg">11</a>
    </span>
    <span class="c5 c26">) and</span> 
    <span>is thus</span>
    <span class="c5 c26">positioned</span> 
    <span>to address all</span>
    <span class="c5 c26">emergent -omics use cases, including Next-Generation Sequencing (NGS). The</span>
    <span>se tools</span>
    <span class="c5 c26"></span>
    <span>are</span>
    <span class="c5 c26">simple to implement, will optimize payload sizes, an</span>
    <span class="c5">d help developers avoid redundant retrieval of data.</span> 
    <span></span>
    <span>They</span>
    <span class="c5">are fully specified on our</span> 
    <span class="c5 c47">
      <a class="c11"
      href="https://www.google.com/url?q=http://genomics-advisor.smartplatforms.org:4000/sequence.html&amp;sa=D&amp;ust=1468821991866000&amp;usg=AFQjCNGDImGPyyaJOH7tKH1mJvXyC0uzJA">
      staging site</a>
    </span>
    <sup>
      <a href="#cmnt9" id="cmnt_ref9">[i]</a>
    </sup>
    <span class="c5">, and</span> 
    <span class="c5 c51 c26">
      <a class="c11" href="#h.pkwqa1">Appendix 1</a>
    </span>
    <span class="c5">of this document shows how DSTU 2.0 can be mapped to the new</span> 
    <span>additions to the resource</span>
    <span class="c5">.</span></p>
    <p class="c12 c9" id="h.1y810tw"></p>
    <h1 class="c71 c9 c53" id="h.tiqdg0uomfty"></h1>
    <ol class="c56 lst-kix_list_4-0" start="3">
      <li class="c31 c9">
        <h1 id="h.4i7ojhp" style="display:inline">
          <span class="c28">Sequence Resource</span>
        </h1>
      </li>
    </ol>
    <p class="c16 c9">
      <span class="c33">
        <a class="c11"
        href="https://www.google.com/url?q=http://hl7-fhir.github.io/sequence.html&amp;sa=D&amp;ust=1468821991871000&amp;usg=AFQjCNF5A9V7Q4znJPsAoxf9cFwO62MupQ">
        http://hl7-fhir.github.io/sequence.html</a>
      </span>
    </p>
    <h2 class="c9 c18" id="h.5kz6hkbnk4j0">
    <span>Structure</span> 
    <span>Diagram</span></h2>
{% include StructureDefinition-Sequence-diff.xhtml %}
    <p class="c22"></p>
    <h2 class="c9 c18" id="h.kvs1hd8l4xje">
      <span>Description</span>
    </h2>
    <p class="c16 c9">
      <span>Sequence resource is designed for next-generation sequencing data. Patients' observed sequences should be represented
      by recording reference sequence id/string and detected variants.</span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9">
    <span class="c5">An overview follows with full details at the</span> 
    <span class="c5 c25">
      <a class="c11"
      href="https://www.google.com/url?q=http://genomics-advisor.smartplatforms.org:4000/sequence-definitions.html&amp;sa=D&amp;ust=1468821991876000&amp;usg=AFQjCNF187j62SzGcLiA1CwOu_fpiXiN7Q">
      staging site</a>
    </span>
    <sup>
      <a href="#cmnt13" id="cmnt_ref13">[m]</a>
    </sup>
    <span class="c5">.</span></p>
    <p class="c16 c9">
    <span class="c5 c35">Sequence.</span>
    <span class="c35">referenceSeq</span>
    <span class="c5">:</span> 
    <span>Four</span>
    <span class="c5">optional</span> 
    <span>ways are provided to represent reference sequence in Sequence resource:</span></p>
    <p class="c16 c9">
      <span>1. Sequence.referenceSeq.referenceSeqId: Reference sequence id in public database;</span>
    </p>
    <p class="c16 c9">
    <span>2.</span> 
    <span>Sequence.referenceSeq.referenceSeqString</span>
    <span>: Reference sequence string</span>
    <span>;</span></p>
    <p class="c16 c9">
      <span>3. Sequence.referenceSeq.referenceSeqPointer: Reference to observed sequence stored in another sequence entity;
      </span>
    </p>
    <p class="c16 c9">
      <span>4. Sequence.referenceSeq.genomeBuild, Sequence.reference.chromosome: The combination of genome build and
      chromosome.</span>
      <span></span>
    </p>
    <p class="c16 c9">
    <span>The w</span>
    <span>indow selects a range from the</span> 
    <span>reference sequence (or genome) that is used to define building block of a current sequence (e.g. Sequence resource
    instance 1).</span></p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9">
    <span class="c5 c35">Sequence.quality</span>
    <span class="c5">: Quality scores for bases in the sequence. Is is intended to be compliant with emerging regulat</span>
    <span>ory needs</span> 
    <span class="c5">needs (eg: those found at PrecisionFDA</span>
    <span>,</span> 
    <span class="c33 c5">
      <a class="c11"
      href="https://www.google.com/url?q=https://precision.fda.gov&amp;sa=D&amp;ust=1468821991885000&amp;usg=AFQjCNFiHL4FEtCBqbJxyHml4JcUUkKMPg">
      https://precision.fda.gov</a>
    </span>
    <span class="c5">).</span></p>
    <p class="c16 c9">
      <span class="c35">Sequence.variant</span>
      <sup>
        <a href="#cmnt14" id="cmnt_ref14">[n]</a>
      </sup>
      <span>: This complex element is used for encoding sequence. When the information of reference sequence and variants are
      provided the observed sequence</span>
      <span class="c35">will</span>
      <span>be derived.</span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9">
      <span class="c35">(Internal Pointers)</span>
    </p>
    <p class="c16 c9">
    <span class="c35">Sequence.patient:</span> 
    <span>This element points to a Patient identifier to show that this sequence is related to the same patient.</span></p>
    <p class="c16 c9">
      <span class="c35">Sequence.specimen</span>
      <span>: A pointer to specimen identifier, if needed.</span>
    </p>
    <p class="c16 c9">
      <span class="c35">Sequence.device</span>
      <span>: A pointer to Device identifier which is used for describing sa equencing method (such as chip id, chip manufacturer
      etc.)</span>
    </p>
    <p class="c16 c9">
      <span class="c35">Sequence.pointer</span>
      <span>: A pointer to a Sequence instance for the next sequence block to build a sequence graph.</span>
    </p>
    <p class="c16 c9">
    <span class="c35">Sequence.observation</span>
    <span>: A pointer to the Observation instance which contains an</span> 
    <span>actionable interpretation for the sequence and the variant found within this sequence block</span>
    <sup>
      <a href="#cmnt15" id="cmnt_ref15">[o]</a>
    </sup>
    <span>.</span></p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9">
      <span class="c35">(External Pointers)</span>
    </p>
    <p class="c16 c9">
    <span class="c5 c35">Sequence.repository</span>
    <span class="c5">:</span>
    <span></span>
    <span class="c5">This complex element is used to provide</span> 
    <span>a</span> 
    <span class="c5">clarifying structure, a base URL, and/or relevant IDs when referring to an external repository.</span></p>
    <p class="c16 c9 c52">
    <span class="c5 c35">GA4GH Repository Example</span>
    <span class="c5">. If the Sequence resource refers to a GA4GH repository for read info, references to a GA4GH full sequence
    dataset should conform to GA4GH</span> 
    <span class="c5 c51">
      <a class="c11"
      href="https://www.google.com/url?q=http://ga4gh.org/%23/documentation&amp;sa=D&amp;ust=1468821991896000&amp;usg=AFQjCNFvvPdDXeV4uD7G7uLN21dgJ66G7Q">
      data models</a>
    </span>
    <span class="c5">and accessed via the GA4GH</span>
    <span class="c5 c25">
      <a class="c11"
      href="https://www.google.com/url?q=http://ga4gh.org/%23/api/v0.5.1&amp;sa=D&amp;ust=1468821991896000&amp;usg=AFQjCNEYUHaJDpvL2H7WAmYoLuXvo7fWvw">API</a>
    </span>
    <span class="c5">. The UR</span>
    <span>L</span>
    <span class="c5">of a GA4GH repository, ids of a GA4GH variant and read group are contained in the Sequence resource. The
    UR</span>
    <span>L</span>
    <span class="c5">of a GA4GH repository is an api_base of a GA4GH server that could be called for sequence data. The GA4GH
    variant set is a collection of call sets and the GA4GH call set is a collection of variant calls, typically for one sample. A
    variant call represents a determination of genotype with respect to that variant.</span></p>
    <p class="c16 c9 c52 c22"></p>
    <p class="c9 c86 c42 c52">
    <span>VariantSet definition:</span> 
    <span>A VariantSet is a collection of variants and variant calls intended to be analyzed together.
    <br /></span></p>
    <p class="c9 c42 c52 c86">
    <span>CallSet definition:</span> 
    <span>A CallSet is a collection of calls that were generated by the same analysis of the same sample.</span></p>
    <p class="c16 c9 c52 c22"></p>
    <p class="c16 c9 c52">
      <span>A read group is a collection of reads produced by a sequencer. A read group set typicall</span>
      <span class="c5">y models reads corresponding to one sample, sequenced one way, and aligned one way. The API reference of
      Google Genomics (</span>
      <span class="c5 c25">
        <a class="c11"
        href="https://www.google.com/url?q=https://cloud.google.com/genomics/v1beta2/reference/&amp;sa=D&amp;ust=1468821991901000&amp;usg=AFQjCNHMiVugAuY0r6C-F_65CUwoS7jBAw">link</a>
      </span>
      <span class="c5">) is a GA4GH repository built by Google and provides details of the data models, such as the resource
      representations.</span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9">
      <span class="c19 c35">Sequence Graph</span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 468.00px;">

        <img alt="Sequence Graph.jpg" src="images/image15.jpg"
        style="width: 624.00px; height: 468.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
      <sup>
        <a href="#cmnt16" id="cmnt_ref16">[p]</a>
      </sup>
      <span></span>
    </p>
    <p class="c9">
      <span>The diagram above describes 4 optional ways provided in the Sequence resource to encode sequencing data.</span>
    </p>
    <p class="c9 c22"></p>
    <p class="c16 c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 468.00px;">

        <img alt="sequence_graph_2.jpg" src="images/image34.jpg"
        style="width: 624.00px; height: 468.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9">
      <span class="c19 c35">Usage Examples</span>
    </p>
{% include Sequence-example-html.xhtml %}
<p>
Source: <a href="Sequence-example.xml.html">XML</a>, <a href="Sequence-example.json.html">JSON</a>, <a href="Sequence-example.ttl.html">Turtle</a>.
</p>
    <p class="c22" id="h.1pxezwc"></p>
    <ol class="c56 lst-kix_list_4-0" start="4">
      <li class="c31 c9">
        <h1 id="h.2jz02w7jszxc" style="display:inline">
          <span class="c28">Observation-genetics Profile</span>
        </h1>
      </li>
    </ol>

{% include StructureDefinition-observationforgenetics-diff.xhtml %}    
    <p class="c22"></p>
    <h2 class="c9 c18" id="h.oprdlzht8czx">
      <span>Description</span>
    </h2>
    <p class="c16 c9" id="h.2p2csry">
      <span class="c5">The Observation-genetics profile is used to describe a</span>
      <span></span>
      <span class="c5">sequence variant</span>
      <span>.</span>
    </p>
    <ul class="c56 lst-kix_t7qwyvm9772b-0 start">
      <li class="c29 c9 c39">
      <span class="c5">The observation-geneticsSequence extension will refer to the Sequence resource for</span> 
      <span>sequence</span>
      <span class="c5">information re</span>
      <span>lated to this variant</span>
      <span class="c5">.</span></li>
      <li class="c29 c9 c39">
        <span>The observation-geneticsInterpretation extension will refer to an Observation instance which contains clinical
        interpretations for the variant described.</span>
      </li>
      <li class="c29 c9 c39">
        <span class="c5">The code, effective[x], issued, performer, method, specimen elements can be used to describe how the
        genetic observation (variant and sequence data) is obtained.</span>
      </li>
      <li class="c29 c9 c39">
        <span>Other extensions are used to describe attributes of this variant such as Genomics Source Class, Amino Acid Change
        Type, etc.</span>
      </li>
      <li class="c29 c9 c39">
        <span>The Observation.result element is used to show whether this variant is detected in this Patient.</span>
      </li>
    </ul>
    <p class="c9 c39 c22"></p>
    <p class="c9 c39 c22"></p>
    <p class="c9 c39">
    <span>Additional Observations instance will be created for variant's further analysis. For example, Observation.component
    element will be used for knowledge-based interpretations of the sequence variant.</span> 
    <span class="c5">Here are some examples for the component.code:</span></p>
    <p class="c16 c9 c22"></p>
    <a id="t.66bb6e456f3a5bfe6405092d619283d1525812e6"></a>
    <a id="t.0"></a>
    <table class="c77">
      <tbody>
        <tr class="c17">
          <td class="c38" colspan="1" rowspan="1">
            <p class="c15 c9 c45">
              <span class="c5 c50 c35">LOINC Code</span>
            </p>
          </td>
          <td class="c41" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5 c50 c35">LOINC Element Name</span>
            </p>
          </td>
          <td class="c48" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5 c35 c50">Comments</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c38" colspan="1" rowspan="1">
            <p class="c15 c9 c45">
              <span class="c5">51963-7</span>
            </p>
          </td>
          <td class="c41" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Medication Assessed</span>
            </p>
          </td>
          <td class="c48" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">A coded medication accessed in a pharmacogenetic test (recommend RxNorm).</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c38" colspan="1" rowspan="1">
            <p class="c15 c9 c45">
              <span class="c5">51967-8</span>
            </p>
          </td>
          <td class="c41" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Genetic disease assessed</span>
            </p>
          </td>
          <td class="c48" colspan="1" rowspan="1">
            <p class="c16 c9 c21 c45">
              <span class="c5">A coded disease that is associated with the region of DNA covered by the genetic test (recommend
              SNOMED).</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c38" colspan="1" rowspan="1">
            <p class="c15 c9 c45">
              <span class="c5">53037-8</span>
            </p>
          </td>
          <td class="c41" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Genetic Disease Sequence Variati</span>
              <span>ant</span>
              <span class="c5">Interpretation</span>
            </p>
          </td>
          <td class="c48" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Interpretation of the pathogenicity of the DNA Sequence Vari</span>
              <span>ant</span>
              <span class="c5">in the context of the assessed genetic disease.</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c38" colspan="1" rowspan="1">
            <p class="c15 c9 c45">
              <span class="c5">53040-2</span>
            </p>
          </td>
          <td class="c41" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Drug Metabolism Sequence Variati</span>
              <span>ant</span>
              <span class="c5">Interpretation</span>
            </p>
          </td>
          <td class="c48" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Predicted phenotype for drug efficacy. A sequence vari</span>
              <span>ant</span>
              <span class="c5">interpretation value known to allow (responsive) or prevent (resistant) the drug to perform.</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c38" colspan="1" rowspan="1">
            <p class="c15 c9 c45">
              <span class="c5">51961-1</span>
            </p>
          </td>
          <td class="c41" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Drug Efficacy Sequence Variati</span>
              <span>ant</span>
              <span class="c5">Interpretation</span>
            </p>
          </td>
          <td class="c48" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Predicted phenotype for ability of drug to bind to intended site in order to deliver intended
              effect. A Sequence Vari</span>
              <span>ant</span>
              <span class="c5">interpretation value known to allow (responsive) or prevent (resistant) the drug to perform.</span>
            </p>
            <p class="c16 c9 c45 c21 c22"></p>
          </td>
        </tr>
      </tbody>
    </table>
    <p class="c16 c9">
      <span>In the meantime, the related element in this Observation instance will point to Observaiton-genetics profile to show
      these clinical interpretations are further analysis for the variant.</span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9">
      <span class="c19 c35">Usage Example</span>
    </p>
    <p class="c16 c9">
      <span>1. An Observation-genetics instance and its two related Observation instances for interpretations</span>
    </p>
    <p class="c16 c9">
      <span>A. Observation-genetics instance which records a variant detected in the patient.</span>
    </p>

{% include Observation-example-genetics-1-html.xhtml %}

<p>
Source: <a href="SObservation-example-genetics-1.xml.html">XML</a>, <a href="Observation-example-genetics-1.json.html">JSON</a>, <a href="Observation-example-genetics-1.ttl.html">Turtle</a>.
</p>


    <p class="c16 c9 c21 c22" id="h.d2v3tv3p3fsi"></p>
    <p class="c16 c9 c21" id="h.8la6qc9vzewl">
      <span>B. An Observation instance which records knowledge-based clinical interpretations for the variant represented in
      A.</span>
    </p>

{% include Observation-example-genetics-2-html.xhtml %}
<p>
Source: <a href="SObservation-example-genetics-2.xml.html">XML</a>, <a href="Observation-example-genetics-2.json.html">JSON</a>, <a href="Observation-example-genetics-2.ttl.html">Turtle</a>.
</p>

    <p class="c16 c9 c21 c22" id="h.im128a39lmy2"></p>
    <p class="c16 c9 c21" id="h.oi5k5ts7ohcx">
      <span>C. An Observation instance which records PCR validation test for the variant in A.</span>
    </p>

    {% include Observation-example-genetics-3-html.xhtml %}
<p>
Source: <a href="SObservation-example-genetics-3.xml.html">XML</a>, <a href="Observation-example-genetics-3.json.html">JSON</a>, <a href="Observation-example-genetics-3.ttl.html">Turtle</a>.
</p>


    <p class="c16 c9 c21 c22" id="h.ihv636"></p>
    <ol class="c56 lst-kix_list_4-0" start="5">
      <li class="c9 c31">
        <h1 id="h.32hioqz" style="display:inline">
          <span class="c28">DiagnosticReport-genetics Profile</span>
        </h1>
      </li>
    </ol>

{% include StructureDefinition-reportforgenetics-diff.xhtml %}    
    
    <p class="c16 c9 c22" id="h.8nlawsyrjho6"></p>
    <p class="c16 c9" id="h.h7zui1epwvmo">
      <span class="c19 c35">Description</span>
    </p>
    <p class="c16 c9" id="h.6o5ts6t8rif3">
      <span class="c5">This DiagnosticReport-genetics is built on top of DiagnosticReport. The new profile is used to describe a
      genetics test report. The result element in DiagnosticReport will refer to the Observation resource that can lead to a bundle
      of genetic observations. And the element of code, effective[x], issued, performer, request, specimen are be used to describe
      the details of the genetic test. Extensions about AssessedCondition and FamilyMemberHistory are added. Overall, this profile
      extends the DiagnosticReport resource to enable reporting of structured genetic test results. In addition, it denotes
      condition context for genetic testing, which may influence reported variants and interpretations for large genomic testing
      panels</span>
      <span class="c5 c25">.</span>
    </p>
    <p class="c16 c9 c22" id="h.1hmsyys"></p>
    <p class="c16 c9">
      <span class="c5">The</span>
      <span class="c5">
        <a class="c11"
        href="https://www.google.com/url?q=http://genomics-advisor.smartplatforms.org:4000/observation-genetics-cg-prf.html&amp;sa=D&amp;ust=1468821991991000&amp;usg=AFQjCNFoEnvuW2owKuHKC7y0Swco1MrCiw"></a>
      </span>
      <span class="c5">DiagnosticReport-genetics profile contains two extensions referring to other resources/profiles and one
      complex extension.</span>
    </p>
    <ul class="c56 lst-kix_jphjql4uq7tq-0 start">
      <li class="c29 c9 c34 c42">
        <span class="c5 c35">AssessedCondition</span>
        <span class="c5">is used to denote condition context for genetic testing, which may influence reported variants and
        interpretation for large genomic testing panels. It refers to the Condition resource.</span>
      </li>
      <li class="c29 c9 c42 c34">
        <span class="c5 c35">FamilyMemberHistory</span>
        <span class="c5">are significant health events and conditions for one or more persons related to the patient. It refers to
        the FamilyMemberHistory-Genetic profile.</span>
      </li>
      <li class="c29 c9 c42 c34">
      <span class="c5 c35">Analysis</span>
      <span class="c5">provides knowledge-based interpretation of overall results of tests performed on patients.</span> 
      <span>E</span>
      <span class="c5">xamples for analysis.code:</span></li>
    </ul>
    <p class="c9 c12"></p>
    <p class="c12 c9"></p>
    <a id="t.6b06d7016a5e6073e3e80a02e46ae3499344bdc0"></a>
    <a id="t.1"></a>
    <table class="c77">
      <tbody>
        <tr class="c17">
          <td class="c38" colspan="1" rowspan="1">
            <p class="c15 c9 c45">
              <span class="c5 c50 c35">LOINC Code</span>
            </p>
          </td>
          <td class="c54" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5 c50 c35">LOINC Element Name</span>
            </p>
          </td>
          <td class="c7" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5 c50 c35">Comments</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c38" colspan="1" rowspan="1">
            <p class="c15 c9 c45">
              <span class="c5">51968-6</span>
            </p>
          </td>
          <td class="c54" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Genetic Disease Analysis Overall Interpretation</span>
            </p>
          </td>
          <td class="c7" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Interpretation of all identified DNA Sequence</span>
              <span>v</span>
              <span class="c5">aria</span>
              <span>nts</span>
              <span class="c5">along with any known clinical information for the benefit of aiding clinicians in understanding the
              results overall in either the context of diagnosis or increased risk of disease.</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c38" colspan="1" rowspan="1">
            <p class="c15 c9 c45">
              <span class="c5">51964-5</span>
            </p>
          </td>
          <td class="c54" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Drug Efficacy Analysis Overall Interpretation</span>
            </p>
          </td>
          <td class="c7" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Overall predicted phenotype for drug efficacy for all DNA Sequence Variants identified in a single
              case.</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c38" colspan="1" rowspan="1">
            <p class="c15 c9 c45">
              <span class="c5">51971-0</span>
            </p>
          </td>
          <td class="c54" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Drug metabolism analysis overall interpretation</span>
            </p>
          </td>
          <td class="c7" colspan="1" rowspan="1">
            <p class="c16 c9 c45 c21">
              <span class="c5">Overall predicted phenotype for drug metabolism for all DNA Sequence Variants identified in a single
              case.</span>
            </p>
          </td>
        </tr>
      </tbody>
    </table>
    <p class="c16 c9 c21 c22" id="h.vx1227"></p>
    <h2 class="c9 c18 c34" id="h.3fwokq0">
      <span class="c19">Usage Example</span>
    </h2>
    <p class="c16 c9">
    <span class="c35">Example</span>
    <span>:</span> 
    <span class="c5">A general genetics test report - here a report of ABCB4 sequence analysis</span>
</p>
{% include DiagnosticReport-dg1-html.xhtml %}
<p>
Source: <a href="DiagnosticReport-dg1.xml.html">XML</a>, <a href="DiagnosticReport-dg1.json.html">JSON</a>, <a href="DiagnosticReport-dg1.ttl.html">Turtle</a>.
</p>

    
    <p class="c22"></p>
    <p class="c16 c9">
    <span class="c35">Example</span>
    <span>:</span> 
    <span class="c5">Genetic test report for a patient with FamilyHistory</span>
    </p>

{% include DiagnosticReport-dg2-html.xhtml %}
<p>
Source: <a href="DiagnosticReport-dg2.xml.html">XML</a>, <a href="DiagnosticReport-dg2.json.html">JSON</a>, <a href="DiagnosticReport-dg2.ttl.html">Turtle</a>.
</p>

    
    <p class="c15 c9 c22"></p>
    <h1 class="c71 c9 c53" id="h.6kly7t8w8mr"></h1>
    <ol class="c56 lst-kix_list_4-0" start="6">
      <li class="c31 c9">
        <h1 id="h.4f1mdlm" style="display:inline">
          <span class="c28">DiagnosticOrder-genetics Profile</span>
        </h1>
      </li>
    </ol>
    <h2 class="c9 c18 c34" id="h.19c6y18">
      <span class="c19">Structure Diagram</span>
    </h2>
    <p class="c16 c9">
      <span class="c5">A complex extension is added on top of the DiagnosticOrder resource. Here is the structure of the
      extension:</span>
    </p>


{% include StructureDefinition-orderforgenetics-diff.xhtml %}    
    
    <h2 class="c9 c18" id="h.2u6wntf">
      <span>Description</span>
    </h2>
    <p class="c9">
      <span>To describe an order requested sequence variants detection. User must set up the code for the request and they can also
      refer to the corresponding sequence instance for that variant.</span>
    </p>
    <h2 class="c9 c18 c34" id="h.3tbugp1">
      <span class="c19">Usage Example</span>
    </h2>
    <p class="c16 c9">
      <span class="c35">Example:</span>
      <span></span>
      <span class="c5">An order for genetics test</span>
      <span>.</span>
    </p>
    <p class="c16 c9">
    <span class="c5">Here is a diagnostic order for testing 185delAG variant. The mother of the patient</span> 
    <span>received results from</span>
    <span class="c5">a mutation panel (eg. MyRisk from Myriad) and she has a BRCA1 185delAG mutation.</span> 
    <span>T</span>
    <span class="c5">he clinician the would like</span> 
    <span>to request</span>
    <span class="c5">to test the patient only for an 185delAG mutation. In this case, the diagnostic order for the patient will
    specify the sequence variant - 185delAG.</span></p>
    <p class="c16 c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 540.00px;">

        <img alt="" src="images/image36.png"
        style="width: 624.00px; height: 540.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c16 c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 278.67px;">

        <img alt="" src="images/image06.png"
        style="width: 624.00px; height: 278.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c22"></p>
    <p class="c22" id="h.28h4qwu"></p>
    <ol class="c56 lst-kix_list_4-0" start="7">
      <li class="c31 c9">
        <h1 id="h.nmf14n" style="display:inline">
          <span class="c28">HLA genotyping results Profile</span>
          <sup>
            <a href="#cmnt17" id="cmnt_ref17">[q]</a>
          </sup>
          <sup>
            <a href="#cmnt18" id="cmnt_ref18">[r]</a>
          </sup>
          <sup>
            <a href="#cmnt19" id="cmnt_ref19">[s]</a>
          </sup>
          <sup>
            <a href="#cmnt20" id="cmnt_ref20">[t]</a>
          </sup>
          <sup>
            <a href="#cmnt21" id="cmnt_ref21">[u]</a>
          </sup>
        </h1>
      </li>
    </ol>
    <h2 class="c9 c18 c34" id="h.37m2jsg">
      <span class="c5 c73 c87">HLA genotyping results profile:</span>
    </h2>

    <p class="c9 c22"></p>
    <p class="c9">
      <span>Human leukocyte antigen (HLA) genotyping is fundamental for research and clinical practice in immunogenetics and
      histocompatibility. Pointers to external locations refer to registered methods, raw NGS reads, and reference standards can be
      conveyed in this profile. Information about and allele assignment is stored in extensions.</span>
    </p>
    <p class="c9">
    <span>The structure of the HLA typing report in this profile attempts to follow the principles outlined in the</span> 
    <span class="c35">M</span>
    <span>inimum</span> 
    <span class="c35">I</span>
    <span>nformation for</span> 
    <span class="c35">R</span>
    <span>eport</span>
    <span class="c35">I</span>
    <span>ng</span> 
    <span class="c35">N</span>
    <span>ext-generation sequence</span> 
    <span class="c35">G</span>
    <span>enotyping (MIRING). These principles were identified through a series of meetings by an international group of
    stakeholders in the application of Next Generation Sequencing (NGS) technology for genotyping the HLA and KIR loci as well as
    other immune-related loci (</span>
    <span class="c33">
      <a class="c11"
      href="https://www.google.com/url?q=http://igdawg.org/ngs.html&amp;sa=D&amp;ust=1468821992071000&amp;usg=AFQjCNHgeqq0pPNggWRRt-QkXpnFt2WnOQ">http://igdawg.org/ngs.html</a>
    </span>
    <span>). MIRING describes eight principles, described in detail in</span> 
    <span class="c33">
      <a class="c11"
      href="https://www.google.com/url?q=http://www.ncbi.nlm.nih.gov/pubmed/26319908&amp;sa=D&amp;ust=1468821992071000&amp;usg=AFQjCNGq-WkqDjS-94QqNdDB4gyMTW4Ukg">
      Human Immunology. 2015 Dec; 76(12):954-962,</a>
    </span>
    <span>. These include detailed metadata about:</span></p>
    <ol class="c56 lst-kix_gzjq6lzg1wwb-0 start" start="1">
      <li class="c29 c9">
        <span>MIRING Annotation</span>
      </li>
      <li class="c29 c9">
        <span>Reference Context</span>
      </li>
      <li class="c9 c29">
        <span>Full Genotype</span>
      </li>
      <li class="c29 c9">
        <span>Consensus Sequence</span>
      </li>
      <li class="c29 c9">
        <span>Novel Polymorphisms</span>
      </li>
      <li class="c29 c9">
        <span>Platform Documentation</span>
      </li>
      <li class="c29 c9">
        <span>Read Processing Documentation</span>
      </li>
      <li class="c29 c9">
        <span>Primary Data</span>
      </li>
    </ol>
    <p class="c9">
    <span>(see the</span> 
    <span class="c33">
      <a class="c11"
      href="https://www.google.com/url?q=http://www.ncbi.nlm.nih.gov/pubmed/26319908&amp;sa=D&amp;ust=1468821992076000&amp;usg=AFQjCNF0qyHgR97R5LMfQ-bdnryB9wdlsg">
      publication above</a>
    </span>
    <span>for details about each)</span></p>
    <p class="c9">
    <span>These principles were implemented in a technical specification by extending an existing XML based format for exchanging
    histocompatibility and immunogenetic genotyping data called Histoimmunogenetics Markup Language (HML) to include results from
    NGS methodologies (</span>
    <span class="c33">
      <a class="c11"
      href="https://www.google.com/url?q=https://bioinformatics.bethematchclinical.org/hla-resources/hml/&amp;sa=D&amp;ust=1468821992077000&amp;usg=AFQjCNGCFbyA5WBHlMLY0_azP5nKLEDitA">https://bioinformatics.bethematchclinical.org/hla-resources/hml/</a>
    </span>
    <span>). The resulting schema may be found int</span> 
    <span class="c33">
      <a class="c11"
      href="https://www.google.com/url?q=https://schemas.nmdp.org/&amp;sa=D&amp;ust=1468821992078000&amp;usg=AFQjCNF7mnzNKpXjbdcvhc8qSKO7OKcmHA">
      schemas.nmdp.org</a>
    </span>
    <span>. The National Marrow Donor Program (NMDP)/Be The Match uses this format for reporting HLA genotyping from potential
    donors and for patients needing stem cell transplants.</span></p>
    <p class="c9 c22"></p>
    <p class="c9">
      <span>Both MIRING and HML were used to inform mapping data elements to FHIR resources such as Patient, Specimen, Sequence,
      ObservationforGenetics Profile, and DiagnosticReport where possible. Several additional data elements were needed
      specifically for this use case, and so a specific profile for reporting HLA genotyping results was developed (Diagnostic
      Report Profile for HLA Genotyping Results).</span>
    </p>
    <ul class="c56 lst-kix_9hu4a59rs9m9-0 start">
      <li class="c9 c80">
      <span class="c62">Minimum information for reporting next generation sequence genotyping (MIRING): Guidelines for reporting
      HLA and KIR genotyping via next generation sequencing</span>
      <span>(see</span> 
      <span class="c50 c47">
        <a class="c11"
        href="https://www.google.com/url?q=http://www.ncbi.nlm.nih.gov/pubmed/26407912&amp;sa=D&amp;ust=1468821992080000&amp;usg=AFQjCNF296EVB9OtBrY-f8d7ZTZb8LVPLw">
        here</a>
      </span>
      <span>)</span></li>
      <li class="c29 c9">
      <span class="c62">Histoimmunogenetics Markup Language 1.0: Reporting Next Generation Sequencing-based HLA and KIR
      Genotyping</span>
      <span>(see</span> 
      <span class="c50 c47">
        <a class="c11"
        href="https://www.google.com/url?q=http://www.ncbi.nlm.nih.gov/pubmed/26319908&amp;sa=D&amp;ust=1468821992081000&amp;usg=AFQjCNGbvoZ7nalbpg10aVOWwFx1Q3hQpQ">
        here</a>
      </span>
      <span>)</span></li>
    </ul>
    <p class="c9 c78 c21 c22"></p>
    <h3 class="c9 c74 c53 c78 c21" id="h.2lwamvv">
      <span class="c5 c28">Structure Diagram of HLA genotyping results Profile</span>
    </h3>


{% include StructureDefinition-hla-genotyping-resultsHaploid-diff.xhtml %}    
        
    <p class="c9">
      <span>Both HLA-genotyping-resultsGlstring and hla-genotyping-resultsHaploid are complex extensions.</span>
    </p>
    <p class="c10 c9">
      <span>The development of these extensions were informed from the allele-assignment structure found in HML. While allele
      assignment for individual loci can be reported in an Observation, here they are used to summarize the assignments at a report
      level. Two methods may be used for reporting HLA allele-assignments: Haploid and GL String.</span>
    </p>
    <p class="c9 c53 c22"></p>
    <p class="c9 c53">
      <span>Structure of hla-genotyping-resultsHaploid:</span>
    </p>

{% include StructureDefinition-hla-genotyping-resultsHaploid-diff.xhtml %}    
    
    <p class="c10 c9">
      <span>Haploid reporting is a method that is used to report an individual allele, or a list of possible alleles if the results
      are ambiguous, using NMDP Multiple Allele Codes (MAC) which is a shorthand for describing allele ambiguity (</span>
      <span class="c33">
        <a class="c11"
        href="https://www.google.com/url?q=https://bioinformatics.bethematchclinical.org/hla-resources/allele-codes/&amp;sa=D&amp;ust=1468821992087000&amp;usg=AFQjCNGAVrZPzuXzkI9Il0hAoSm_nN9eMw">https://bioinformatics.bethematchclinical.org/hla-resources/allele-codes/</a>
      </span>
      <span>). MACs are widely used for reporting HLA typing results because they are able to report long allele lists in a small
      amount of space, but the system has a number of shortcomings (</span>
      <span class="c33">
        <a class="c11"
        href="https://www.google.com/url?q=http://www.ncbi.nlm.nih.gov/pubmed/23849068&amp;sa=D&amp;ust=1468821992088000&amp;usg=AFQjCNEcZ7Or7IAAHJd0Xfrf_KeFtfdKMA">Tissue
        Antigens. 2013 Aug;82(2):106-12</a>
      </span>
      <span>). When reporting data using haploid, typical use is one or two haploid elements for a particular locus, but possibly
      more if multiple loci are covered (ex: two HLA-DRB1 haploids + one HLA-DRB3 haploid).</span>
    </p>
    <p class="c10 c9 c22"></p>
    <p class="c10 c9">
      <span>Within Haploid, Method indicates whether the general methodology is DNA based typing (e.g., Sequence Specific Primers
      (SSP), Sequence Specific Oligonucleotide Probes (SSOP), or Sequence Based Typing (SBT)), or SER which indicates serology
      based methods.</span>
    </p>
    <p class="c10 c9 c22"></p>
    <p class="c10 c9">
      <span>Example of using two hla-genotyping-resultsHaploid extensions to report a HLA-A genotype of
      <br />HLA-A*01:AB and HLA-A*02:MN which expands to</span>
    </p>
    <p class="c10 c9">
      <span>HLA-A*01:01/HLA-A*01:02 and HLA-A:02:01/HLA-A:02:02/HLA-A:02:03</span>
    </p>
    <p class="c10 c9 c22"></p>
    <a id="t.d2654932bcf278aba9dc2d00d192ef3c0f97f95b"></a>
    <a id="t.2"></a>
    <table class="c57">
      <tbody>
        <tr class="c17">
          <td class="c70" colspan="1" rowspan="1">
            <p class="c10 c9">
              <span class="c6">&lt;extension url=&quot;http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsHaploid&quot;
              &gt;</span>
            </p>
            <p class="c20 c9">
              <span class="c6">  &lt;extension url=locus&gt;</span>
            </p>
            <p class="c9 c10">
              <span class="c6">    &lt;valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;coding&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;system value=&quot;http://www.genenames.org&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;code value=&quot;4931&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;display value=&quot;HLA-A&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;/coding&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;text value=&quot;HLA-A&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;/valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">  &lt;extension url=type&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;coding&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;system
              value=&quot;https://bioinformatics.bethematchclinical.org/hla-resources/allele-codes/&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;code value=&quot;AB&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;display value=&quot;01/02&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;/coding&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;text value=&quot;HLA-A*01:AB&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;/valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">  &lt;extension url=method&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;text value=DNA/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;/valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">&lt;/extension&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">&lt;extension url=&quot;http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsHaploid&quot;
              &gt;</span>
            </p>
            <p class="c20 c9">
              <span class="c6">  &lt;extension url=locus&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;coding&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;system value=&quot;http://www.genenames.org&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;code value=&quot;4931&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;display value=&quot;HLA-A&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;/coding&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;text value=&quot;HLA-A&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;/valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">  &lt;extension url=type&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;coding&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;system
              value=&quot;https://bioinformatics.bethematchclinical.org/hla-resources/allele-codes/&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;code value=&quot;AB&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">        &lt;display value=&quot;01/02/03&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;/coding&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;text value=&quot;HLA-A*02:MN&quot;/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;/valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">  &lt;extension url=method&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">      &lt;text value=DNA/&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">    &lt;/valueCodeableConcept&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c10 c9">
              <span class="c6">&lt;/extension&gt;</span>
            </p>
          </td>
        </tr>
      </tbody>
    </table>
    <p class="c10 c9 c22"></p>
    <p class="c10 c9 c22"></p>
    <p class="c10 c9">
      <span>Because of limitations with MAC, another method called GL Strings was developed that encodes the results in a text
      string with hierarchical set of operators to describe the relationships between alleles, lists of possible alleles, phased
      alleles, genotypes, lists of possible genotypes, and multilocus unphased genotypes, without losing typing information or
      increasing typing ambiguity. (</span>
      <span class="c33">
        <a class="c11"
        href="https://www.google.com/url?q=http://www.ncbi.nlm.nih.gov/pubmed/23849068&amp;sa=D&amp;ust=1468821992125000&amp;usg=AFQjCNEYC3pHBGIDS8BmODLeR4HKZL9BdQ">Tissue
        Antigens. 2013 Aug;82(2):106-12</a>
      </span>
      <span>).</span>
    </p>
    <p class="c9 c21 c22"></p>
    <p class="c9">
      <span>The structure of HLA-genotyping-resultsGlstring:</span>
    </p>

{% include StructureDefinition-hla-genotyping-resultsGlstring-diff.xhtml %}    

    
    <p class="c10 c9">
    <span>While t</span>
    <span>he string format is easily parsed into separate components to be rendered for user viewing,</span> 
    <span class="c46">GL Strings by themselves are potentially quite long and difficult to read. It often advantageous to point to
    a URI which may return the GL String on demand to avoid manual data entry. This is available through the URI element in this
    extension.</span></p>
    <p class="c9">
      <span>Example of using the hla-genotyping-resultsGlstring to report a GL String in both text and URI formats</span>
    </p>
    <a id="t.54f6fe1d8e36e93b4b9fac661d95ee55699b9e9c"></a>
    <a id="t.3"></a>
    <table class="c57">
      <tbody>
        <tr class="c17">
          <td class="c70" colspan="1" rowspan="1">
            <p class="c20">
              <span class="c6">&lt;extension
              url=&quot;http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsGlstring&quot;&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;extension url=text&gt;  </span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;valueString
              value=&quot;HLA-A*01:01:01:01/HLA-A*01:02+HLA-A*02:01:02/HLA-A*02:02:02/HLA-A*02:03:01&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;extension url=uri&gt;</span>
            </p>
            <p class="c20">
              <span class="c75">    &lt;valueUri value=&quot;</span>
              <span class="c33 c75">
                <a class="c11"
                href="https://www.google.com/url?q=https://gl.nmdp.org/imgt-hla/3.23.0/genotype/1h%2522/&amp;sa=D&amp;ust=1468821992138000&amp;usg=AFQjCNGg0BAGtYuBSqYOwrrJAaLRgMupzg">https://gl.nmdp.org/imgt-hla/3.23.0/genotype/1h&quot;/</a>
              </span>
              <span class="c6">&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">&lt;/extension&gt;</span>
            </p>
          </td>
        </tr>
      </tbody>
    </table>
    <p class="c9 c22"></p>
    <p class="c9 c21">
      <span>Whether reporting in Haploid or GL String formats, it is important to identify the version of the IMGT/HLA allele
      database, as new HLA alleles are constantly being discovered and allele assignment is based on the known alleles at the time,
      and so results may need to be reinterpreted later. This is done through the Allele Database element.</span>
    </p>
    <p class="c9 c53">
      <span>Structure of hla-genotyping-resultsAlleleDatabase</span>
    </p>

{% include StructureDefinition-hla-genotyping-resultsAlleleDatabase-diff.xhtml %}    
    
    <p class="c9">
      <span>Example of using hla-genotyping-resultsAlleleDatabase to report using the IMGT/HLA database, version 3.23</span>
    </p>
    <a id="t.968b7039e2c4c82645f1620d268120918e3f6bf8"></a>
    <a id="t.4"></a>
    <table class="c57">
      <tbody>
        <tr class="c17">
          <td class="c70" colspan="1" rowspan="1">
            <p class="c20">
              <span class="c6">&lt;extension
              url=&quot;http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsAlleleDatabase&quot;&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;valueCodeableConcept&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;system value=&quot;http://www.ebi.ac.uk/ipd/imgt/hla/&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;version value=&quot;3.23&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;/coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;text value=&quot;IMGT/HLA 3.23&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/valueCodeableConcept&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">&lt;/extension&gt;</span>
            </p>
          </td>
        </tr>
      </tbody>
    </table>
    <p class="c9 c22"></p>
    <p class="c10 c9 c22"></p>
    <p class="c9 c53">
      <span>Structure of hla-genotyping-resultsAlleleAssignmentDate</span>
    </p>
Error: this is not valid: {.% include StructureDefinition-hla-genotyping-resultsAlleleAsignmentDate-diff.xhtml %.}    

    <sup>
      <a href="#cmnt22" id="cmnt_ref22">[v]</a>
    </sup>
    <p class="c10 c9 c22"></p>
    <p class="c9">
      <span>Example of hla-genotyping-resultsAlleleAsignmentDate</span>
    </p>
    <a id="t.aae56bb89f4c3ba7777a65033423b9936ff97b57"></a>
    <a id="t.5"></a>
    <table class="c57">
      <tbody>
        <tr class="c17">
          <td class="c70" colspan="1" rowspan="1">
            <p class="c20">
              <span class="c6">&lt;extension
              url=&quot;http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsAlleleAssignmentDate&quot;&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;valueDate value=&quot;2016-06-01&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">&lt;/extension&gt;</span>
            </p>
          </td>
        </tr>
      </tbody>
    </table>
    <p class="c10 c9 c22"></p>
    <p class="c10 c9">
      <span>Overall methodology may be reported using hla-genotyping-resultsMethod. Here a codeable concept may be used to refer to
      method entered into a public registry, such as the NCBI Genetic Test Registry, or a local private registry.</span>
    </p>
    <p class="c10 c9 c22"></p>
    <p class="c10 c9">
      <span>Structure of hla-genotyping-resultsMethod</span>
    </p>

{% include StructureDefinition-hla-genotyping-resultsMethod-diff.xhtml %}    

    <p class="c9">
      <span>Example of hla-genotyping-resultsMethod to report a lab test registered in the NCBI Genetic Test Registry</span>
    </p>
    <a id="t.35527ccfea688e361930d0551da68bd282edff21"></a>
    <a id="t.6"></a>
    <table class="c57">
      <tbody>
        <tr class="c17">
          <td class="c70" colspan="1" rowspan="1">
            <p class="c20">
              <span class="c6">&lt;extension
              uri=&quot;http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsMethod&quot;&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;valueCodeableConcept&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">     &lt;coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c75">       &lt;system value=&quot;</span>
              <span class="c75 c33">
                <a class="c11"
                href="https://www.google.com/url?q=http://www.ncbi.nlm.nih.gov/gtr/%2522/&amp;sa=D&amp;ust=1468821992159000&amp;usg=AFQjCNF9eE99u7uA43ypTx7Sl6cOWCmZaQ">http://www.ncbi.nlm.nih.gov/gtr/&quot;/</a>
              </span>
              <span class="c6">&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">       &lt;code value=GTR000000000.0&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">     &lt;/coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">     &lt;text value=NGS based HLA-A genotyping</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/valueCodeableConcept&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">&lt;/extension&gt;</span>
            </p>
          </td>
        </tr>
      </tbody>
    </table>
    <p class="c9 c22"></p>
    <p class="c9">
      <span>Example of an typing report</span>
    </p>
    <ul class="c56 lst-kix_yefcnq5qqs1-0 start">
      <li class="c29 c9">
        <span>for HLA-A, HLA-B, HLA-C loci,</span>
      </li>
      <li class="c29 c9">
        <span>by Sequence Based Typing (SBT) of exons 2 and 3,</span>
      </li>
      <li class="c29 c9">
        <span>using Next Generation Sequencing technologies,</span>
      </li>
      <li class="c29 c9">
        <span>allele assignment based on the IMGT/HLA database version 3.23,</span>
      </li>
      <li class="c29 c9">
        <span>reported multilocus unphased genotype using GL Strings.</span>
      </li>
    </ul>
    <p class="c9 c22"></p>
    <a id="t.6dbb7d2cdbfb87e6b88c3e4141d42420eff9ee88"></a>
    <a id="t.7"></a>
    <table class="c57">
      <tbody>
        <tr class="c17">
          <td class="c70" colspan="1" rowspan="1">
            <p class="c20">
              <span class="c6">&lt;DiagnosticReport&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;text&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">   &lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;h3&gt;HLA-A, -B, -C Genotyping Report for Marry Chalmers (MRN: 12345) &lt;/h3&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;pre&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    Gene   Allele 1         Allele 2</span>
            </p>
            <p class="c20">
              <span class="c6">    HLA-A   HLA-A*01:01:01:01     HLA-A*01:02</span>
            </p>
            <p class="c20">
              <span class="c6">    HLA-B   HLA-B*15:01:01:01     HLA-B*57:01:01</span>
            </p>
            <p class="c20">
              <span class="c6">    HLA-C   HLA-C*01:02:01      HLA-C*03:04:01:01</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;/pre&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;p&gt;Allele assignments based on IMGT/HLA 3.23&lt;/p&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;p&gt;Issued: 2015-05-26T15:30:10+01:00&lt;/p&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;p&gt;Method: Next-Generation Sequencing of exons 2 and 3 of HLA Class I
              genes&lt;/p&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;p&gt;Lab: ACME labs, Inc&lt;/p&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">   &lt;/div&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/text&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;code&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;system value=&quot;http://loinc.org&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;code value=&quot;53767-0&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;display value=&quot;HLA-A+B+C SBT [Interpretation] by High
              resolution&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;/coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/code&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;category&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;system
              value=&quot;http://hl7.org/fhir/ValueSet/diagnostic-service-sections&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;code value=&quot;GE&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;display value=&quot;Genetics&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;/coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/category&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;subject&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;reference value=&quot;Patient/123&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;display value=&quot;Marry Chalmers&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/subject&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;specimen&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;reference value=&quot;Specimen/456&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;display value=&quot;buccal swab&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/specimen&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;effectiveDateTime value=&quot;2015-05-26T15:30:10+01:00&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;issued value=&quot;2014-05-16T10:28:00+01:00&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;extension xmlns=&quot;http://hl7.org/fhir&quot;</span>
            </p>
            <p class="c20">
              <span class="c6">   
              url=&quot;http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsAlleleAsignmentDate&quot;&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;valueDate value=&quot;2014-05-16T10:28:00+01:00&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;extension xmlns=&quot;http://hl7.org/fhir&quot;</span>
            </p>
            <p class="c20">
              <span class="c6">   
              url=&quot;http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsAlleleDatabase&quot;&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;valueCodeableConcept&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">        &lt;system value=&quot;http://www.ebi.ac.uk/ipd/imgt/hla/&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">        &lt;version value=&quot;3.23&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;/coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;text value=&quot;IMGT/HLA 3.23&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;/valueCodeableConcept&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;extension xmlns=&quot;http://hl7.org/fhir&quot;</span>
            </p>
            <p class="c20">
              <span class="c6">    url=&quot;http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsGlstring&quot;
              &gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;extension url=&quot;text&quot;&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;valueString
              value=&quot;HLA-A*01:01:01:01+HLA-A*01:02^HLA-B*15:01:01:01+HLA-B*57:01:01^HLA-C*01:02:01+HLA-C*03:04:01:01&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;/extension&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;extension url=&quot;uri&quot;&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;valueUri
              value=&quot;https://gl.nmdp.org/imgt-hla/3.23.0/multilocus-unphased-genotype/i&quot; /&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;/extension&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;extension xmlns=&quot;http://hl7.org/fhir&quot;</span>
            </p>
            <p class="c20">
              <span class="c6">   
              url=&quot;http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsMethod&quot;&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;valueCodeableConcept&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">        &lt;system value=&quot;http://www.ncbi.nlm.nih.gov/gtr/&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">        &lt;code value=&quot;GTR000000000.0&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;/coding&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">      &lt;text value=&quot;Next Generation Sequencing of exons 2 and 3 of HLA Class I
              genes&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;/valueCodeableConcept&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/extension&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;result&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;reference value=&quot;Observation/111&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;display value=&quot;Genotype Observation for for HLA-A&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/result&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;result&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;reference value=&quot;Observation/222&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;display value=&quot;Genotype Observation for for HLA-B&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/result&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;result&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;reference value=&quot;Observation/333&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">    &lt;display value=&quot;Genotype Observation for for HLA-C&quot;/&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">  &lt;/result&gt;</span>
            </p>
            <p class="c20">
              <span class="c6">&lt;/DiagnosticReport&gt;</span>
            </p>
            <p class="c20 c22"></p>
          </td>
        </tr>
      </tbody>
    </table>
    <p class="c9 c22"></p>
    <p class="c9">
      <span>In the above example, three Observations are referenced, each containing the details surrounding the genotyping of each
      gene/locus. Those observations would further refer to other observations (using Observation.related) that point to phased
      exon sequences which are the basis for individual allele assignment. This strategy is illustrated in the following figure for
      one gene, and sequencing two exons for each allele. Note that references from DiagnosticReport and Observation back to
      Patient/Donor and Specimen are not shown for clarity, although we would include them in the resource instances.</span>
    </p>
    <p class="c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 396.00px;">

        <img alt="" src="images/image25.png"
        style="width: 624.00px; height: 396.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <h2 class="c9 c18 c34" id="h.3l18frh">
      <span class="c19">Usage Example</span>
    </h2>
    <p class="c9 c43 c21 c22"></p>
    <p class="c9 c43 c21">
      <span class="c5">If the developer wants to retrieve the HLA genotyping report of patient 123, the API call would be:</span>
    </p>
    <p class="c16 c9">
      <span class="c8 c5">GET /DiagnosticReport?</span>
    </p>
    <p class="c16 c9">
      <span class="c8 c5">profile=hla-genotyping-results&amp;</span>
    </p>
    <p class="c16 c9">
      <span class="c8 c5">subject=123</span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9 c22"></p>
    <hr style="page-break-before:always;display:none;" />
    <p class="c16 c9 c22"></p>
    <p class="c16 c9 c22"></p>
    <ol class="c56 lst-kix_list_4-0" start="8">
      <li class="c31 c9">
        <h1 id="h.tf76y3w5j4lx" style="display:inline">
          <span class="c28">Relationship among Sequence resource and genetics profiles</span>
        </h1>
      </li>
    </ol>
    <p class="">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 428.00px;">

        <img alt="" src="images/image00.png"
        style="width: 624.00px; height: 428.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c22"></p>
    <h2 class="c53" id="h.39l6umf1zexg">
      <span>Pointers to Sequence resource</span>
    </h2>
    <p class="">
      <span>Sequence resource is used to transmit and represent sequencing data. In FHIR Genomics, there are several genetics
      profiles containing internal pointers to Sequence resource for genetic data reporting. Here is the summary:</span>
    </p>
    <p class="">
      <span class="c35">Sequence.pointer</span>
      <span>: A bundle of Sequence instances can be connected by Sequence.pointer to build sequence graph described in Section 3.
      Sequence.pointer will point to the next sequence block.</span>
    </p>
    <p class="">
      <span class="c35">Sequence extension in Observation-genetics profile</span>
      <span>: Observation-genetics profile is used to report a genetic variant found in patients. Sequence extension contains a
      pointer to Sequence identifier which has related sequencing read info. It will provide reference/observed allele information
      and quality scores for each based/sequence block.</span>
    </p>
    <p class="c22"></p>
    <h2 class="c53" id="h.x0nn4mltj1t">
      <span>Pointer to Observation resource (based on genetics profile)</span>
    </h2>
    <p class="">
      <span>Observation-genetics based Observation resource is used for interpretative genetic data. Sequence resource and
      genetics profiles will use internal pointers to Observation-genetics-profile based Observation instance for variant report.
      </span>
    </p>
    <p class="">
      <span class="c35">Sequence.observation</span>
      <span>: A pointer to genetics Observation instance which is used for interpretations of this sequence block. Interpretations
      are generally about genetic variant found is this sequence block.</span>
    </p>
    <p class="">
      <span class="c35">DiagnosticReport.result</span>
      <span>: A pointer to Observation instance. This element is used for reporting genetic result.</span>
    </p>
    <p class="">
      <span class="c35">Item extension in DiagnosticOrder-genetics profile</span>
      <span>: It is used to describe the genetic test order.</span>
    </p>
    <p class="">
      <span class="c35">FamilyMemberHistory</span>
      <span>: A pointer to genetics Observation instance to report genetic test results of family member</span>
    </p>
    <p class="c22"></p>
    <h2 class="c53" id="h.xom3kxq6nzsk">
      <span>Observation vs. Sequence</span>
    </h2>
    <p class="">
    <span>Sequence is used for raw sequencing data while genetic-profile-based Observation resource is used for reporting
    interpretative genetic information (eg. DNA/amino acid variant detected in the patient). Although both Sequence and
    Observation-genetics profiles contain variant info, they are used differently: Sequence.variant is used for encoding the
    sequence block; the Observation-genetics profile is mainly used for a variants test result or</span> 
    <span>interpretations</span>
    <span>of raw data in Sequence resource.</span></p>
    <p class="c22"></p>
    <h2 class="c53" id="h.7o3xmvc75lx">
      <span>Usage example</span>
    </h2>
    <p class="">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 457.33px;">

        <img alt="Screen Shot 2016-07-12 at 9.14.25 PM.png" src="images/image09.png"
        style="width: 624.00px; height: 457.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c9 c42 c61 c21">
      <span>This use case depicts the whole story of FHIR in the clinical genomics setting: an order is requested, then test
      results are reported. A Sequence instance and two Observation instances for further interpretation are also created.</span>
    </p>
    <p class="c22"></p>
    <ol class="c56 lst-kix_list_4-0" start="9">
      <li class="c31 c9">
        <h1 id="h.76drlbxj2vl8" style="display:inline">
          <span class="c28">HL7 Domain Analysis Use Cases</span>
        </h1>
      </li>
    </ol>
    <p class="">
    <span>Simulated use of the proposed Sequence resource and profiles on other FHIR resources are shown for many of the examples
    in</span> 
    <span class="c35">Section 5  Use Case Scenarios</span>
    <span>in the</span> 
    <span class="c35">HL7 Domain Analysis Model</span> 
    <span class="c35">(DAM): Clinical Genomics, Release 1, September 2014 Informative Ballot</span>
    <span>(</span>
    <span class="c33">
      <a class="c11"
      href="https://www.google.com/url?q=https://www.hl7.org/documentcenter/public_temp_649E1787-1C23-BA17-0CA4C354EB3E41D5/wg/clingenomics/docs/V3DAM_CG_CLINSEQ_R1_O1_2013JAN.pdf&amp;sa=D&amp;ust=1468821992239000&amp;usg=AFQjCNG6QS0IohMn1js4MBHF1cmevFhn9g">link</a>
    </span>
    <span>).</span></p>
    <p class="">
      <span>We show use of the FHIR design we are proposing for these cases:</span>
    </p>
    <p class="c52">
      <span>1.Specimen Identification</span>
    </p>
    <p class="c76">
      <span>1.1Get references to all variants obtained from germline analysis.</span>
    </p>
    <p class="c76">
      <span>1.2Explore antibacterial drug resistance over time (non-patient DNA)</span>
    </p>
    <p class="c52">
      <span>2.Clinical Sequencing  Germline Testing</span>
    </p>
    <p class="c52">
      <span>3.Cancer Profiling</span>
    </p>
    <p class="c52">
      <span>4Decision Making Tools  Family History and Drug Dosage Calculators</span>
    </p>
    <p class="c76">
      <span>4.1Today clinicians translate (i.e. manually re-enter) genetic data into tools for decision making. This
      includes family history tools and drug dosage calculators. In the future, this data will automatically be incorporated into
      clinical decision making tools.</span>
    </p>
    <p class="c42 c45 c59 c61 c76">
      <span>4.2Get pharmacogenomic reporting and disease risk or diagnosis associated with variant</span>
    </p>
    <p class="c30">
      <span>5.Public Health</span>
    </p>
    <p class="c30">
      <span>6.</span>
      <span>Clinical and Research Data Warehouses</span>
    </p>
    <p class="c42 c45 c59 c22 c61"></p>
    <p class="c22"></p>
    <p class="c22"></p>
    <h2 class="c9 c18 c34" id="h.xasd2mds301m">
      <span class="c19">1. Specimen Identification</span>
    </h2>
    <ul class="c56 lst-kix_list_1-0 start">
      <li class="c29 c16 c9">
        <span class="c5">Germline testing for biomarkers/mutations (usually inherited)</span>
      </li>
      <li class="c29 c16 c9">
        <span class="c5">Tumor testing for biomarkers/mutations (somatic / tumor specific)</span>
      </li>
      <li class="c29 c16 c9">
        <span class="c5">Pediatric Testing</span>
      </li>
      <li class="c29 c16 c9">
        <span class="c5">Infectious Disease Testing</span>
      </li>
    </ul>
    <h3 class="c9 c74 c53 c34 c21" id="h.1egqt2p">
      <span class="c5 c28">Subcase: Get references to all variants obtained from germline analysis.</span>
    </h3>
    <p class="c9 c43">
      <span class="c8 c5">GET</span>
      <span class="c32"> </span>
      <span class="c8 c5">/</span>
      <span class="c32">Observation?</span>
    </p>
    <p class="c9 c43">
      <span class="c32">        _profile=</span>
      <span class="c32">http://hl7.org/fhir/StructureDefinition/observationforgenetics&amp;</span>
    </p>
    <p class="c9 c43">
      <span class="c32">        GenomicSourceClass=http://hl7.org/fhir/LOINC-48002-0-answerlist|LA6683-2</span>
    </p>
    <p class="c15 c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 362.67px;">

        <img alt="" src="images/image30.png"
        style="width: 624.00px; height: 362.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c16 c9">
      <span class="c5">*Return a bundle of genetic</span>
      <span>s-profile-based Observation</span>
      <span class="c5">instances</span>
    </p>
    <h3 class="c9 c53 c34 c21 c60" id="h.5q9pxmwi736c"></h3>
    <p class="c22"></p>
    <p class="c22" id="h.2dlolyb"></p>
    <h2 class="c9 c18 c34" id="h.sqyw64">
      <span class="c19">2. Clinical Sequencing  Germline Testing</span>
    </h2>
    <p class="c16 c9">
    <span class="c5">For a chart review,</span> 
    <span>g</span>
    <span class="c5">et references to all DNA sequences related to mutations with</span> 
    <span>a</span>
    <span class="c5">n interpretation Unknown Significance</span></p>
    <p class="c16 c9">
      <span class="c8 c5">GET /</span>
      <span class="c32">Observation</span>
      <span class="c8 c5">?</span>
    </p>
    <p class="c3">
      <span class="c8 c5">subject=123&amp;</span>
    </p>
    <p class="c3">
      <span class="c8 c5">component-code-value-[x]= http://loinc.org|53037-8$LA6682-4</span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c16 c9">
      <span class="c8 c5"></span>
      <span class="c5 c26">53037-8: LOINC code for Genetic disease sequence varia</span>
      <span>nt</span>
      <span class="c5 c26">interpretation</span>
    </p>
    <p class="c16 c9">
      <span class="c5 c26">LA6682-4: LOINC answer code for Unknown Significance</span>
      <span>.</span>
    </p>
    <p class="c9 c15">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 461.33px;">

        <img alt="" src="images/image04.png"
        style="width: 624.00px; height: 461.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c22" id="h.3cqmetx"></p>
    <h2 class="c9 c18 c34 c44" id="h.yoimsumxv1jj"></h2>
    <hr style="page-break-before:always;display:none;" />
    <h2 class="c9 c18 c44 c34" id="h.jv825xsbulv5"></h2>
    <h2 class="c9 c18 c34" id="h.1rvwp1q">
      <span class="c19">3. Cancer Profiling</span>
    </h2>
    <p class="c16 c9">
    <span>The goal of this profile methodology is to g</span>
    <span class="c5">et references</span> 
    <span>from</span>
    <span class="c5">all variants obtained from somatic analysis.</span>
    <span></span>
    <span class="c5">Changes in the population of cells with particular mutations will change overtime as well as in conjunction
    with events such as therapy. For instance, targeted chemotherapy may kill a specific population of cancer cells with specific
    mutations and other cancer cell populations may survive and continue to divide. Therefore, clearly annotating these specimens
    as somatic variants and capturing annotations related to a time relevant to a treatment timeline may be critical for
    analysis.</span></p>
    <p class="c9 c43">
      <span class="c8 c5">GET /</span>
      <span class="c32">Observation?</span>
    </p>
    <p class="c9 c43">
      <span class="c32">       _profile=</span>
      <span class="c32">http://hl7.org/fhir/StructureDefinition/observationforgenetics&amp;</span>
    </p>
    <p class="c9 c13">
      <span class="c32">GenomicsS</span>
      <span class="c8 c5">ourceClass=http://hl7.org/fhir/LOINC-48002-0-answerlist|LA6684-0&amp;</span>
    </p>
    <p class="c13 c9">
      <span class="c5 c8">date=2015-07-04&amp;</span>
    </p>
    <p class="c13 c9">
      <span class="c8 c5">subject=123</span>
    </p>
    <p class="c9 c43 c55">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 400.00px;">

        <img alt="" src="images/image37.png"
        style="width: 624.00px; height: 400.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <hr style="page-break-before:always;display:none;" />
    <p class="c22"></p>
    <p class="c22" id="h.4bvk7pj"></p>
    <h2 class="c9 c18 c34" id="h.2r0uhxc">
      <span class="c19">4. Decision Making Tools  Family History and Drug Dosage Calculators</span>
    </h2>
    <h3 class="c9 c74 c53 c34 c21" id="h.1664s55">
      <span class="c5 c28">4.1 Today clinicians translate (i.e. manually re-enter) genetic data into tools for decision making.
      This includes family history tools and drug dosage calculators. In the future, this data will automatically be incorporated
      into clinical decision making tools.</span>
    </h3>
    <p class="c16 c9">
      <span class="c5">Get family history related to one observation:</span>
    </p>
    <p class="c3">
      <span class="c8 c5">GET /</span>
      <span class="c32">DiagnosticReport</span>
      <span class="c8 c5">?</span>
    </p>
    <p class="c3">
      <span class="c32">       _profile=</span>
      <span class="c32">http://hl7.org/fhir/StructureDefinition/reportforgenetics&amp;</span>
    </p>
    <p class="c3">
      <span class="c8 c5"></span>
      <span class="c32">subject</span>
      <span class="c8 c5">=123</span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c15 c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 604.00px; height: 309.00px;">

        <img alt="Screen Shot 2016-04-30 at 4.35.10 PM.png" src="images/image02.png"
        style="width: 604.00px; height: 309.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c22"></p>
    <p class="" id="h.3q5sasy">
      <span></span>
    </p>
    <h2 class="c9 c53" id="h.25b2l0r">
      <span>4.2 Get pharmacogenomic reporting and disease risk or diagnosis associated with variant</span>
    </h2>
    <p class="c3">
      <span class="c8 c5">GET /</span>
      <span class="c32">Observation?</span>
    </p>
    <p class="c9 c78">
      <span class="c32">      
      _profile=http://hl7.org/fhir/StructureDefinition/observationforgenetics&amp;</span>
    </p>
    <p class="c3 c52">
      <span class="c8 c5">subject=123&amp;</span>
    </p>
    <p class="c3 c52">
      <span class="c32">Interpretation.</span>
      <span class="c8 c5">component-code= http://loinc.org|51963-7</span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c3">
      <span class="c5">51963-7 : LOINC code for Medication Assessed </span>
    </p>
    <p class="c15 c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 480.00px;">

        <img alt="Screen Shot 2016-04-30 at 5.04.20 PM.png" src="images/image08.png"
        style="width: 624.00px; height: 480.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <hr style="page-break-before:always;display:none;" />
    <p class="c22"></p>
    <p class="c22" id="h.kgcv8k"></p>
    <h3 class="c9 c53 c34 c21 c74" id="h.34g0dwd">
      <span class="c19">5. Public Health</span>
    </h3>
    <p class="c16 c9">
      <span class="c5">Today Registrars manually translate clinical data into public health reporting systems. This data is used to
      monitor and improve public health (e.g. surveillance and clinical research). In the future, this data will be extracted from
      the EHR in an automated (or semi-automated) f</span>
      <span>ashion</span>
      <span class="c5">.</span>
    </p>
    <p class="c16 c9">
    <span class="c5">For a breast cancer clinical genomic study,</span> 
    <span>g</span>
    <span class="c5">et all genetic-profile-based observations of patients with breast cancer:</span></p>
    <p class="c16 c9 c22"></p>
    <p class="c3">
      <span class="c8 c5">GET /</span>
      <span class="c32">DiagnosticReport</span>
      <span class="c8 c5">?</span>
    </p>
    <p class="c3">
      <span class="c32">       _profile=</span>
      <span class="c32">http://hl7.org/fhir/StructureDefinition/reportforgenetics&amp;</span>
    </p>
    <p class="c3">
      <span class="c8 c5"></span>
      <span class="c32">A</span>
      <span class="c8 c5">ssessed</span>
      <span class="c32">C</span>
      <span class="c8 c5">ondition.code=http://snomed.info/sct|254837009</span>
    </p>
    <p class="c16 c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 620.00px; height: 444.00px;">

        <img alt="Screen Shot 2016-05-09 at 11.06.58 AM.png" src="images/image29.png"
        style="width: 620.00px; height: 444.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c16 c9 c22"></p>
    <p class="c22" id="h.1jlao46"></p>
    <h3 class="c9 c60 c53 c34 c21" id="h.iuoa1fmucstv"></h3>
    <hr style="page-break-before:always;display:none;" />
    <h3 class="c9 c60 c53 c34 c21" id="h.8blmu3zdfzq0"></h3>
    <h3 class="c9 c74 c53 c34 c21" id="h.43ky6rz">
      <span class="c19">6. Clinical and Research Data Warehouses</span>
    </h3>
    <p class="c16 c9">
      <span class="c5">Health data warehousing should persist data in its standardized formats, while allowing users to export
      subsets of the data in the warehouse into multiple data marts', optimized for specific use cases, analysis type or reporting
      needs.</span>
    </p>
    <p class="c16 c9">
      <span class="c5">Get all genetic-profile-based observations of patients with the variant c.181T&gt;G</span>
    </p>
    <p class="c3">
      <span class="c8 c5">GET /Observation?</span>
    </p>
    <p class="c13 c9 c52">
      <span class="c32">_</span>
      <span class="c8 c5">p</span>
      <span class="c32">rofile=</span>
      <span class="c32">http://hl7.org/fhir/StructureDefinition/observationforgenetics</span>
      <span class="c32">&amp;</span>
    </p>
    <p class="c3 c52">
      <span class="c32">DNAV</span>
      <span class="c8 c5">ariantID=http://www.ncbi.nlm.nih.gov/projects/SNP|rs58238560</span>
    </p>
    <p class="c9 c43 c21">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 364.00px;">

        <img alt="Screen Shot 2016-05-09 at 11.24.29 AM.png" src="images/image07.png"
        style="width: 624.00px; height: 364.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c16 c9 c22"></p>
    <hr style="page-break-before:always;display:none;" />
    <p class="c22"></p>
    <p class="c22"></p>
    <ol class="c56 lst-kix_list_4-0" start="10">
      <li class="c31 c9">
        <h1 id="h.1dg3bpe69muw" style="display:inline">
          <span class="c28">Other Genomics Use Cases</span>
        </h1>
      </li>
    </ol>
    <p class="">
      <span>During the development of the FHIR Genomics design, CGWG participants have commented on the earlier use cases and/or
      proposed new use cases. In this section, we list describe of these new cases and demonstrate how the proposed design will
      address them.</span>
    </p>
    <h4 class="c9 c53" id="h.xvir7l">
      <span class="c19">1. Get a patient's family members genetics report.</span>
    </h4>
    <p class="c16 c9">
      <span>This example is proposed by Kevin Hughes. Family history is useful for clinicians to know more about the condition of
      the patient.</span>
    </p>
    <p class="c3">
      <span class="c8 c5">Get /FamilyMemberHistory?</span>
    </p>
    <p class="c3 c52">
      <span class="c32">_profile=</span>
      <span class="c32">http://hl7.org/fhir/StructureDefinition/familymemberhistory-genetic</span>
      <span class="c32">&amp;</span>
    </p>
    <p class="c3 c52">
      <span class="c8 c5">patient=123</span>
    </p>
    <p class="c15 c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 292.00px;">

        <img alt="Screen Shot 2016-05-09 at 11.52.45 AM.png" src="images/image22.png"
        style="width: 624.00px; height: 292.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c16 c9 c22"></p>
    <h4 class="c9 c53 c82" id="h.71ezdtx13k34"></h4>
    <h4 class="c9 c18 c83 c45 c59" id="h.wg6bgvykahmq">
      <span class="c19">2. Panel</span>
    </h4>
    <p class="c9 c43">
      <span class="c5">Search for results from nephrotic syndrome panel</span>
    </p>
    <p class="c3">
      <span class="c8 c5">Get /DiagnosticReport&amp;</span>
    </p>
    <p class="c3 c52">
      <span class="c32">_profile=</span>
      <span class="c32">http://hl7.org/fhir/StructureDefinition/reportforgenetics</span>
      <span class="c32">&amp;</span>
    </p>
    <p class="c3 c52">
      <span class="c8 c5">code=N0336&amp;</span>
    </p>
    <p class="c3 c52">
      <span class="c8 c5">patient=123</span>
    </p>
    <p class="c15 c9">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 485.50px; height: 248.20px;">

        <img alt="OU?2015-11-17 H3.28.55.png" src="images/image13.png"
        style="width: 485.50px; height: 248.20px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <h4 class="c9 c18 c83 c45 c59" id="h.7cxdzrpwk3ii">
    <span class="c19">3.</span> 
    <span class="c19 c24">Sequence Test</span> 
    <span class="c19">Metrics</span></h4>
    <a id="id.wiuj2ermofm3"></a>
    <p class="c22"></p>
    <h4 class="c9 c18 c82 c45 c59 c83" id="h.3hnqitwfkb2y"></h4>
    <p class="c3 c52 c22"></p>
    <p class="c16 c9 c22"></p>
    <ol class="c56 lst-kix_list_4-0" start="11">
      <li class="c31 c9">
        <h1 id="h.hb7rvdbqv2xg" style="display:inline">
          <span>Clinico-Genomic Apps</span>
        </h1>
      </li>
    </ol>
    <p class="">
      <span>The idea for a Sequence resource grew out, in part, the SMART Platforms Project, which explored creating
      clinico-genomic apps to integrated traditional EMR clinical data and genomic data to show data visualization and analysis,
      including CDS that depended upon both types of data. Below are a couple of examples. Several apps have already been
      designed including Genomics Advisor, SMART Precision Cancer Medicine, and Diabetes Bear EMR. Below, one of these apps will
      be described. To include other apps in this section, please feel free to add a note on it and how it uses FHIR/Genomics
      calls. </span>
    </p>
    <ol class="c56 lst-kix_oai74q1k79ui-0 start" start="1">
      <li class="c63 c53 c69">
        <h2 id="h.pwa28js4v2x8" style="display:inline">
          <span>Genomics Advisor</span>
        </h2>
      </li>
    </ol>
    <p class="c42 c45 c59 c61">
    <span>T</span>
    <span>he SMART on FHIR Genomics Advisor was an app incorporating genomics data to show risk of disease, drug susceptibility,
    and related conditions based upon genotype. Technically, this app was architected (see below) by combining data from
    independent data services, a SMART on FHIR clinical server for clinical information and one for a SMART on FHIR Genomics data
    server for genomic</span> 
    <span>data</span>
    <span>. The set of FHIR API calls that are necessary to support this app are shown below:</span></p>
    <p class="c42 c45 c59 c61 c22"></p>
    <p class="c55">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 503.49px; height: 240.48px;">

        <img alt="OU?2015-11-16 H12.56.26.png" src="images/image26.png"
        style="width: 503.49px; height: 240.48px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="">
      <span>The presentation of the app looks like this:</span>
    </p>
    <p class="c55 c58">
      <span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 1.33px solid #cccccc; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 469.00px; height: 269.65px;">

        <img alt="" src="images/image31.png"
        style="width: 542.46px; height: 312.63px; margin-left: -73.46px; margin-top: -32.11px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);"
        title="" />
      </span>
    </p>
    <p class="c22"></p>
    <p class="c22"></p>
    <p class="c22"></p>
    <p class="c22"></p>
    <h1 class="c71 c9 c53" id="h.to3s0xdvt9pc"></h1>
    <hr style="page-break-before:always;display:none;" />
    <h1 class="c71 c9 c53" id="h.vaw81nfschpt"></h1>
    <h1 class="c9 c53 c71" id="h.195x9360uyxh"></h1>
    <h1 class="c9 c74 c53 c79" id="h.pkwqa1">
    <span class="c28">Appendix 1: From DSTU2 Genomics to</span> 
    <span class="c28">STU3</span></h1>
    <p class="c16 c9 c21 c22"></p>
    <p class="c16 c9 c21">
      <span class="c5">The table shows how the data elements in the DSTU2 Observation resource as extended by the Standard Genetics
      Profile would be mapped to the new Sequence resource.</span>
    </p>
    <p class="c16 c9 c21 c22"></p>
    <a id="t.97227904e5be677dba3166652dba74116f802fff"></a>
    <a id="t.8"></a>
    <table class="c68">
      <tbody>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
            <span class="c65 c35 c26">DSTU2</span> 
            <span class="c65 c26">Standard</span>
            <span class="c35 c26 c65">Genetics profile</span></p>
            <p class="c36 c9 c21">
              <span class="c65 c35 c26">on Observation</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Observation-genetics profile</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c65 c35 c26">Maps to Sequence (Proposed)</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsGenomeBuild</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Sequence extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">referenceSeq.genomicsBuild</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsChromosome</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Sequence extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">referenceSeq.chromosome</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsGenomicsStart</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Sequence extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">variant.start</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsGenomicsStop</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Sequence extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">variant.end</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsReferenceAllele</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Sequence extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">variant.referenceAllele</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsObservedAllele</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Sequence extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">variant.observedAllele</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsSpecies</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Sequence extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">specimen</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsAllelicState</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Allelic State extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c9 c21 c36">
              <span class="c1">geneticsAllelicFrequency</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Allelic Frequency extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsReadCoverage</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Sequence extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">readCoverage</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsCopyNumberEvent</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Copy number event extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsTranscriptReferenceSequenceId</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Transcript reference sequence identifier extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsProteinReferenceSequenceId</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Protein reference sequence identifier</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsCIGAR</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Sequence extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">variant.cigar</span>
            </p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsVariationId</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">DNA variation id extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsVariationType</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">DNA sequence variation type extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsAminoAcidChange</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Amino acid change extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsAminoAcidChangeType</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Amino acid change type extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsGene</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Gene extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsDNARegionName</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">DNA region name extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
        <tr class="c17">
          <td class="c27" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">geneticsGenomicSourceClass</span>
            </p>
          </td>
          <td class="c0" colspan="1" rowspan="1">
            <p class="c36 c9 c21">
              <span class="c1">Genomic source class extension</span>
            </p>
          </td>
          <td class="c23" colspan="1" rowspan="1">
            <p class="c36 c9 c21 c22"></p>
          </td>
        </tr>
      </tbody>
    </table>
    <p class="c9 c22" id="h.39kk8xu"></p>
    <div>
      <p class="c66 c42 c45 c81">
      <span class="c35 c47">Page</span> 
      <span class="c24 c47 c49">of</span></p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref1" id="cmnt1">[a]</a>
        <span class="c37 c5">Changed &quot;effort&quot; to &quot;family&quot; (see
        https://www.hl7.org/fhir/comparison.html).</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref2" id="cmnt2">[b]</a>
        <span class="c37 c5">Changed &quot;return&quot; to &quot;hold&quot; because not always the use of a resource is in a
        context of &#39;return&#39; scenarios.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref3" id="cmnt3">[c]</a>
        <span class="c37 c5">Does it make sense to call a resource &quot;Sequence&quot; and at the time time accommodate any type
        of &#39;-omics&#39; data? If the omics is the scope of this resource, why not call it Omics? However, it is important to
        note that the FHIR new resource criteria (http://wiki.hl7.org/index.php?title=Template:FHIR_Resource_Proposal) define as
        must that the new resource &quot;Represents a well understood, &quot;important&quot; concept in the business of
        healthcare&quot;.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref4" id="cmnt4">[d]</a>
        <span class="c37 c5">What other FHIR resources target &quot;undefined types of&quot; data? The only one I can think of is
        the Observation resource that is a simple code-value pair (with metadata) that can carry any value defined by the code
        and/or by external terminologies. If the Observation resource is the &#39;model&#39; for creating a flexible structure,
        then in the case of omics we should rethink the Sequence proposal and possibly focus on something more basic that can
        really be a basis for new &amp; existing types of omics data, e.g., a genetic locus. Then, data sets such as this Sequence
        proposal or other types of omics data could profile the base resource, if it&#39;s truly generic as the Observation
        resource is.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref5" id="cmnt5">[e]</a>
        <span class="c37 c5">This ballot was about &quot;clinical sequencing&quot;.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref6" id="cmnt6">[f]</a>
        <span class="c37 c5">Changed &quot;return&quot; to &quot;hold&quot; (see previous comment).</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref7" id="cmnt7">[g]</a>
        <span class="c37 c5">Changed &quot;tools&quot; to &quot;resource and profiles&quot;. The connotation of tools to some users
        could be more on tools used for the the implementation of these resource and profiles.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref8" id="cmnt8">[h]</a>
        <span class="c37 c5">See previous comment.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref9" id="cmnt9">[i]</a>
        <span class="c37 c5">I thought it was agreed to move to FHIR.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref10" id="cmnt10">[j]</a>
        <span class="c37 c5">The Sequence proposal as it&#39;s presented in the current build
        (http://hl7-fhir.github.io/sequence.html) doesn&#39;t meet the FHIR new resource criterion for &quot;high cohesion&quot;
        (see http://wiki.hl7.org/index.php?title=Template:FHIR_Resource_Proposal). It&#39;s a humongous mixed-bag of raw sequence
        data (ACGT...) along reference sequences, variants, structure variants, quality data and more.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref11" id="cmnt11">[k]</a>
        <span class="c37 c5">I might make a suggestion to try and limit screen shots and repeating of info that can be found on the
        FHIR resource pages. As we make changes, a screen shot will be immediately out of date. It seems OK to me to rely on the
        FHIR resource to cover the basics, and this IG should be higher level &quot;here is how you use it&quot; types of
        information.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref12" id="cmnt12">[l]</a>
        <span class="c37 c5">Why cardinality of variant is 0..* and structure variant only 0..1?</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref13" id="cmnt13">[m]</a>
        <span class="c37 c5">See previous comment.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref14" id="cmnt14">[n]</a>
        <span class="c37 c5">See comment below on the &#39;sequence graph&#39;.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref15" id="cmnt15">[o]</a>
        <span class="c37 c5">The purpose of the variant element is defined above as &quot;used for encoding sequence&quot;, but
        here it seems to be a variant found in some analysis that is oriented towards &quot;actionable interpretation&quot;. Such
        variants should be represented solely through the observation genetic profile. The &#39;sequence encoding variants&#39;
        should be removed (see comment below).</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref16" id="cmnt16">[p]</a>
        <span class="c37 c5">This is an attempt to invent yet another way to represent a raw sequence and should be out-of-scope
        for HL7, since it&#39;s in fact a re-modeling of bioinformatics sequence formats in FHIR constructs that are meant to cover
        merely common data. In addition, the observed sequence attribute is a simple string and could lead to confusion if used
        naively.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref17" id="cmnt17">[q]</a>
        <span class="c37 c5">+bmilius@nmdp.org or +jschneid@nmdp.org, you&#39;ll add HLA material, right?</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref18" id="cmnt18">[r]</a>
        <span class="c37 c5">yes, I&#39;ll upload material by the end of day today.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref19" id="cmnt19">[s]</a>
        <span class="c5 c37">uploaded some, still have much to do :(</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref20" id="cmnt20">[t]</a>
        <span class="c37 c5">okay, mostly done. Please read and let me know if you see any issues.</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref21" id="cmnt21">[u]</a>
        <span class="c37 c5">Thanks Bob, I&#39;ll definitely read the section before the end of the day and I&#39;ll let you know
        if I see any issues!</span>
      </p>
    </div>
    <div class="c40">
      <p class="c20">
        <a href="#cmnt_ref22" id="cmnt22">[v]</a>
        <span class="c37 c5">not sure if this is actually needed. DiagnosticReport already has effectiveDateTime and issued
        elements that can do this.</span>
      </p>
    </div>
  </body>
</html>
