<!DOCTYPE HTML>

[%settitle Pharmacogenomic Clinical Decision Support Implementation Guide%]
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
  [%file newheader%]

  <style>

    .code {
        font-family: Consolas,Menlo,Monaco,Lucida Console,Liberation Mono,DejaVu Sans Mono,Bitstream Vera Sans Mono,Courier New,monospace,sans-serif;
        color:black;
        background-color:rgb(250,250,250);
        margin-top: 1em;
        margin-bottom: 1em;
        padding: 10px;
        line-height: 1.5em;
        font-size: 13px;
        overflow: auto;
    }

    .p-l-10 {
        padding-left: 10px;
    }

    .cql-keyword {
        color: rgb(0, 0, 255);
        font-weight: 700;
    }

    .cql-string {
        color: rgb(163, 21, 21);
    }

    .cql-variable, .cql-property {
        color: rgb(128, 0, 0);
    }

    .cql-class, .cql-object {
        color: rgb(25, 127, 157);
    }

    .cql-literal, .cql-operator {
        color: rgb(64,64,64);
    }

    .cql-interface {
        color: rgb(43, 170, 175);
    }

  </style>
</head>
<body>
  [%file newnavbar%]

<div class="col-9">
  <a name="pCDS"/>
  <h3>Pharmacogenomic Clinical Decision Support Implementation Guide</h3>

  <a name="introduction"/>
  <h4>Introduction</h4>

  <p>
    The DIGITizE Action Collaborative (<a href="http://iom.nationalacademies.org/Activities/Research/GenomicBasedResearch/Innovation-Collaboratives/EHR.aspx">more information on DIGITizE</a>) is focused on increasing support for clinical genomics within the electronic health record (EHR) ecosystem.
    It is approaching this broad goal by initially focusing on creating support for a very specific set of use cases.
    These use cases are being used as a means to bring relevant stakeholders together to produce functionality that improves clinical care.
    Our intention is to use learning derived from these initial use cases to incrementally broaden support to additional areas.
    By doing so, we hope to create a framework for broadly expanding health information technology support for genetics and genomics.
  </p>
    <p>
    Our initial use cases focus on two pharmacogenomic effects:  the impact of a specific HLA-B allele on abacavir hypersensitivity and the
    impact of certain TPMT alleles on azathioprine dosing.  In both cases the members of DIGITizE felt that it was optimal to ensure the patient&#39;s
    health record contains the specific qualified genetic observation result for the associated drug before prescribing the drug.
    Therefore, consistent with published Clinical Pharmacogenetics Implementation Consortium (CPIC) clinical guidelines (see reference section),
    we are suggesting the following Clinical Decision Support (CDS) be implemented to warn the clinician in the following situations:
    </p>

  <ul>
  	<li>An order for abacavir or azathioprine is placed for a patient that does not have a record of the appropriate qualified genetic observation result in the EHR environment. </li>
  	<li>An order for abacavir is placed for a patient with the specific HLA-B observation result stating the presence of a specific HLA-B allele, which would indicate whether the patient is likely to have a hypersensitivity reaction.</li>
  	<li>An order for azathioprine is placed for a patient with the specified TPMT observation result indicating that the patient is an intermediate or poor metabolizer of the drug.</li>
  </ul>

  <p>The goal of this guide is to specify how system developers and healthcare organizations can implement this clinical decision support.
    It begins with a proposed standardized process for transmitting the data needed to underlie this support and then follows with a suggestion for how the CDS rules themselves could be implemented.</p>

  <a name="scope"/>
  <h4>Scope</h4>


  <p>This guide is intended to facilitate the rapid implementation of two important pharmacogenomic clinical decision support rules.
    While we believe the suggestions in this guide are extensible to other pharmacogenomic effects, this work has not yet been completed.</p>

  <p>This guide does not cover the transmission of structured data describing genetic variants or positions or regions assayed in tests.
    These transfers are not required to implement the use cases outlined below.  However, we believe these transmissions will be critical to
    many future use cases.  For this reason, we intend to develop a guide that covers the transmission of these data.
    As this is done, it may make sense to update this implementation guide.  Users of this guide should be aware that this future work
    could involve deprecating the structures and strategies proposed below.</p>

    <a name="moving"/>
    <h4>Moving the Data</h4>

  <p>Laboratories that generate genetic test results often run different information systems than the providers who receive these results.
    Therefore it is important that there be a standardized method for transmitting information between these systems.
    The DIGITizE Standards working group has developed a process that can be used to transmit the structured data needed to underlie these
    clinical decision support rules.  While we have primarily focused our attention on transmissions from laboratory information systems (LIS)
    to the EHR environment, this process could also be used to transmit needed data between EHRs or LISs.</p>

  <p>The working group has determined the minimal set of laboratory observation results needed to support these use cases.
    The definition of these results and their controlled vocabulary were established by the CPIC through a consensus-based Delphi survey entitled,
     &quot;CPIC Term Standardization for Clinical Pharmacogenetic Test Results.&quot;  (URL provided in reference section.)</p>

  <p>In the body of this guide we outline a method for moving the required data through HL7 Fast Healthcare Interoperability Resources (FHIR) - See description here: <a href="hl7.org/fhir/DSTU2/observation-genetics-cg-prf-1a.html">hl7.org/fhir/DSTU2/observation-genetics-cg-prf-1a.html</a>
    and example here: <a href="hl7.org/fhir/DSTU2/obs-genetics-example3-mutationlist-3.json.html">hl7.org/fhir/DSTU2/obs-genetics-example3-mutationlist-3.json.html</a>.</p>

  <a name="rationale"/>
  <h5>Rationale</h5>

  <p>Our current goal is to build momentum for the exchange of genetic data, provide guidance for using these data in clinical decision support,
    and gather information to help guide future standards and recommendations. To help achieve this goal, we&#39;ve devised a basic method for collecting
    and exchanging information that we believe will reduce barriers to entry for organizations interested in participating.
    The current method has limitations - for example, it does not support exchanging discrete variant information or other metadata.
    It is not intended to be a long-term solution and we plan to develop new guidance for exchanging data as this initiative progresses.</p>

  <a name="g032"/>
  <h5>Abacavir and HLA-B</h5>
  <p>The CPIC Delphi survey #3 Q7, related to the HLA-B alleles that are strongly associated with specific adverse effects to drugs,
    covers the absence or presence of the lab results for 3 different HLA-B alleles, each with a relationship to an adverse drug reaction for a unique drug.
    Abacavir hypersensitivity is specifically related to the HLA-B allele referred to as HLA-B*57:01.</p>

  <blockquote>
    <p>While this pilot project is not specifically focused on the other two HLA-B alleles covered by CPIC&#39;s
      Delphi survey Q13, which are related to different drugs, it is assumed that laboratory observation results for these other HLA-B allele
      findings could support additional CDS rules related to the specific ADRs associated with those allele findings. Refer to the last section titled
      &quot;Additional Use Cases&quot; for more information on these two HLA-B alleles and their association with adverse drug reactions.</p>
  </blockquote>

  <p>The HLA-B*57:01 [Presence] observation result is defined in the CPIC survey #3 (Q7) to provide a result from a testing laboratory of whether
    the HLA-B*57:01 allele was POSITIVE or NEGATIVE in a tested subject.  No further observation results are required from the testing laboratory
    in order to fulfill the CDS rules for the working groupâ€™s initial use case surrounding abacavir.</p>

  <p>An existing LOINC observation code used for reporting the presence or absence of the HLA-B*57:01 allele is described in more detail in
    the section titled, LOINC Codes, and is expected to be integrated into the sending laboratory&#39;s FHIR test results and consumed by the
    receiving EHR&#39;s system so that it may be stored in a manner that supports the CDS pilot for abacavir.</p>

  <a name="g033"/>
  <h5>Azathioprine and TPMT</h5>
  <p>The CPIC Delphi survey #4 Q4, related to the pharmacogenetic gene phenotypes, covers the controlled vocabulary to be used by labs to
    report the interpreted phenotype based on the results of a pharmacogenetic test performed by the lab for several genes that code for drug
    metabolizing enzymes.  The interpreted phenotype of one of these genes, TPMT, drives the azathioprine use case defined by the working group.
    It should be noted that TPMT gene variants affect mercaptopurine and thioguanine to a similar extent as their effect on azathioprine, so modest
    changes to implementation can be used to extend this approach to all thiopurine drugs.</p>

  <blockquote>
    <p>While this pilot project is not specifically focused on the other drug metabolizing genes included in the
      CPIC survey (DPYD, CYP2D6, CYP2C9, CYP2C19, CYP3A5, UGT1A1) because they are not related to azathioprine, it is assumed that laboratory
      observation results for these other genes could support additional CDS rules related to the specific PGx CDS rules associated with these drug
      metabolizing genes. Refer to the <a href="#g037">Additional Use Cases</a> section for more information on other drug metabolizing enzyme genes.</p>
  </blockquote>

  <p>While this pilot project is not specifically focused on the other drug metabolizing genes included in the
    CPIC survey (DPYD, CYP2D6, CYP2C9, CYP2C19, CYP3A5, UGT1A1) because they are not related to azathioprine, it is assumed that laboratory
    observation results for these other genes could support additional CDS rules related to the specific PGx CDS rules associated with these drug
    metabolizing genes. Refer to the <a href="#g037">Additional Use Cases</a> section for more information on other drug metabolizing enzyme genes.</p>

  <p>The TPMT Gene Product Metabolic Activity Interpretation observation is defined in the CPIC survey #4 (Q4) to provide a result from a testing
    laboratory of one of the specific controlled terms that represent the interpreted phenotype of TPMT as determined by the testing laboratory.
    No further observation results are required from the testing laboratory in order to fulfill the CDS rules for the initial use case surrounding
    azathioprine.</p>

  <p>A new LOINC code is proposed in the following section and is expected to be integrated into the sending laboratory&#39;s FHIR test results and
    consumed by the receiving EHR&#39;s system so that it may be stored in a manner that supports the CDS pilot for azathioprine.</p>

  <a name="g034"/>
  <h5>Loinc Codes</h5>
  <p>The individual laboratory observation result codes that enable the CDS for abacavir and azathioprine are respectively referred to as
    HLA-B*57:01 [Presence] and TPMT Gene Product Metabolic Activity Interpretation.</p>

  <blockquote>
    <p><strong>A note about LOINC answer codes</strong> <br/>LOINC can and does provide a special class of codes that are
      only used as &quot;Answer&quot; codes.  These codes begin with the characters &quot;LA&quot; and will be specified within the definition of a LOINC code for which
      they can be used.  SNOMED-CT codes are the preferred standard.  In the case of the HLA-B*57:01 observation, SNOMED equivalent SNOMED-CT terms
      for Positive and Negative exist and are specified by this guideline as the answer codes that will trigger the associated CDS rules. However,
      the specific answer codes for TPMT gene product metabolic activity interpretation have not yet been established in SNOMED-CT.
      Follow up work may be pursued to establish these codes in SNOMED-CT and which point a modification to this guideline could be made to make use of
      these new codes in addition to the original LOINC answer codes.  <!--It should be noted that multiple &quot;equivalent&quot; codes can be sent in the OBX-5 field
      by separating them with a tilde &quot;~&quot;, which should allow implementers to support one or the other or both answer codes as they become available.-->
      An example of providing the optional LOINC answer codes for the HLA-B*57:01 allele along with the equivalent SNOMED-CT answer codes has been
      provided below.</p>
  </blockquote>

  <a name="g035"/>
  <h5>HLA-B*57:01 [Presence]</h5>
  <p>The existing LOINC code, 50956-2: HLA-B*57:01[Presence], is designed to communicate either the presence or absence of the specific HLA-B*57:01 allele.
    <br/>Here are the details: (<a href="http://s.details.loinc.org/LOINC/50956-2.html?sections=Simple">http://s.details.loinc.org/LOINC/50956-2.html?sections=Simple <img src="../external.png" style="text-align: baseline"/></a>)</p>

  <p>
    When representing this observation in FHIR, we use the Observation resource (<a href="http://hl7-fhir.github.io/observation.html">http://hl7-fhir.github.io/observation.html</a>).
    The type of the observation, 50956-2:HLA-B*57:01 is specified in the "code" element of the Observation (<a href="http://hl7-fhir.github.io/observation-definitions.html#Observation.code">http://hl7-fhir.github.io/observation-definitions.html#Observation.code</a>).
    The result value is specified in the value[x] element, specifically the valueCodeableConcept, using the appropriate SNOMED code for Positive or Negative.
  </p>

   <!-- <p>When applying this observation to an HL7v2 message, 50956-2:HLA-B*57:01 represents the observation code that would populate the OBX-3 field of a
    single OBX segment returned to the ordering provider system from the testing laboratory system.  The observation result value associated with
    this same OBX segment is found in the field OBX-5 which would contain one of two ordinal values, positive or negative.</p>  -->

  <blockquote>
    <p>The SNOMED-CT codes for Positive and Negative are 10828004 and 260385009, respectively.
      SNOMED codes are strongly recommended for accurately and precisely representing the Positive or Negative result.</p>
  </blockquote>

  <p>
    The reporting laboratory is to use this LOINC code to affirm that they tested for the specific HLA-B*57:01 allele and one of the two SNOMED
    codes to convey they have observed that the patient&#39;s specimen either contains it or does not contain it.
    If an observation containing this LOINC code is not returned by a testing laboratory and stored in a manner that is available to the CDS
    algorithm then it cannot be assumed that the patient has been tested for the presence or absence of the HLA-B*57:01 allele, which is directly
    related to the hypersensitivity of abacavir.
    <br/>
  </p>
  <p>
    Here is an example of the key elements discussed above:<br/>
    For a positive finding of the HLA-B*57:01 allele...
    <div class="code">
      <span class="cql-keyword" id="ObservedAllele">define </span><span class="cql-string">"Observed HLA-B*57:01 Allele"</span><span class="cql-generic">:</span>
      <br/>
      <div class="p-l-10">
        <span class="cql-generic">[</span><span class="cql-property">Observation</span><span class="cql-generic">: </span><span class="cql-property">"Genotype Results for HLA-B*57:01 Allele"</span><span class="cql-generic">]</span>
      </div>
      <br/>
      <span class="cql-keyword">define </span><span class="cql-string" id="PositiveHyper">"Positive Status for Hypersensitivity"</span><span class="cql-generic">:</span>
      <br/>
      <div class="p-l-10">
        <a href="#ObservedAllele"><span class="cql-interface" style="text-decoration: underline">"Observed HLA-B*57:01 Allele"</span></a><span class="cql-object"> A</span>
        <br/>
        <div class="p-l-10">
          <span class="cql-keyword">where </span><span class="cql-object">A</span><span class="cql-generic">.</span><span class="cql-literal">value </span><span class="cql-operator">= </span><!--
          --><span class="cql-keyword">Code </span><span class="cql-generic">{ </span> <span class="cql-literal">code</span><span class="cql-generic">: </span><!--
          --><span class="cql-string">'10828004' </span><span class="cql-literal">system</span><span class="cql-generic">: </span><span class="cql-string">"http://snomed.org/sct" </span><!--
          --><span class="cql-literal">display</span><span class="cql-generic">: </span><span class="cql-string">'Positive' </span><span class="cql-generic">}</span>
        </div>
      </div>
    </div>
    And, for a negative finding of the HLA-B*57:01 allele...
    <div class="code">
      <span class="cql-keyword">define </span><span class="cql-string" id="NegativeHyper">"Negative Status for Hypersensitivity"</span><span class="cql-generic">:</span>
      <br/>
      <div class="p-l-10">
        <a href="#ObservedAllele"><span class="cql-interface" style="text-decoration: underline">"Observed HLA-B*57:01 Allele"</span></a><span class="cql-object"> A</span>
        <br/>
        <div class="p-l-10">
          <span class="cql-keyword">where </span><span class="cql-object">A</span><span class="cql-generic">.</span><span class="cql-literal">value </span><span class="cql-operator">= </span><!--
          --><span class="cql-keyword">Code </span><span class="cql-generic">{ </span> <span class="cql-literal">code</span><span class="cql-generic">: </span><!--
          --><span class="cql-string">'260385009' </span><span class="cql-literal">system</span><span class="cql-generic">: </span><span class="cql-string">"http://snomed.org/sct" </span><!--
          --><span class="cql-literal">display</span><span class="cql-generic">: </span><span class="cql-string">'Negative' </span><span class="cql-generic">}</span>
        </div>
      </div>
    </div>

  </p>

  <!-- <p>The reporting laboratory is to use this LOINC code to affirm that they tested for the specific HLA-B*57:01 allele and one of the two SNOMED
    codes to convey they have observed that the patient&#39;s specimen either contains it or does not contain it.
    If an OBX segment containing this LOINC code is not returned by a testing laboratory and stored in a manner that is available to the CDS
    algorithm then it cannot be assumed that the patient has been tested for the presence or absence of the HLA-B*57:01 allele, which is directly
    related to the hypersensitivity of abacavir.
    <br/>Here&#39;s a partial OBX segment example of the key elements discussed above:</p>
    <p>For a positive finding of the HLA-B*57:01 allele...<br/>
    <strong>OBX|1|..|50956-2^HLA-B*57:01^LN ||10828004^Positive^SCT~LA6576-8^Positive^LN-ANS <br/>|...</strong><br/>
    And, for a negative finding of the HLA-B*57:01 allele...<br/><strong>OBX|1|..|50956-2^HLA-B*57:01^LN ||260385009^Negative^SCT~LA6577-6^Negative^LN-ANS <br/>
      |...</strong></p>-->

  <blockquote>
    <p>NOTE: <!--The examples above includes the equivalent SNOMED-CT (SCT) and LOINC Answer (LN-ANS) codes in each Component Code field.-->
      While SNOMED-CT is the preferred standard, it does not prevent the ability to provide the equivalent answer code from the LOINC Answer list.
      This guide does specify a requirement to use the SNOMED-CT values so that the receiving system can fulfill the CDS rules associated with this test
      result.</p>
  </blockquote>

  <!-- <p style="border:2px solid black;">NOTE: The examples above includes the equivalent SNOMED-CT (SCT) and LOINC Answer (LN-ANS) codes in each OBX-5 field.
    While SNOMED-CT is the preferred standard, it does not prevent the ability to provide the equivalent answer code from the LOINC Answer list.
    This guide does specify a requirement to use the SNOMED-CT values so that the receiving system can fulfill the CDS rules associated with this test
    result.</p> -->

  <a name="g036"/>
  <h4><span class="sectioncount">G.0.3.6</span> TPMT Gene Product Metabolic Activity Interpretation <a href="PharmacogenomicSupport-IG.html#g036" title="link to here" class="self-link"><img src="../target.png"/></a></h4>
  <p>A new LOINC observation code, 79713-4: TPMT gene product metabolic activity interpretation , has been created precisely to support the requirement for
    the azathioprine use case.  The details of the LOINC code follow: <a href="http://s.details.loinc.org/LOINC/79713-4.html?sections=Simple">http://s.details.loinc.org/LOINC/79713-4.html?sections=Simple</a>
  </p>

  <p>Here is an example to show how this LOINC code would be applied:</p>

  <p>For an Intermediate metabolizer TPMT gene product metabolic activity interpretation observation...<br/>

    <div class="code">
      <span class="cql-keyword">define </span><span class="cql-string" id="Results">"TPMT Results"</span><span class="cql-generic">:</span>
      <br/>
      <div class="p-l-10">
        <span class="cql-generic">[</span><span class="cql-property">Observation</span><span class="cql-generic">: </span><span class="cql-property">"Genotype Results for TPMT"</span><span class="cql-generic">]</span>
      </div>
      <br/>
      <span class="cql-keyword">define </span><span class="cql-string" id="Intermediate">"Intermediate Status"</span><span class="cql-generic">:</span>
      <br/>
      <div class="p-l-10">
        <a href="#Results"><span class="cql-interface" style="text-decoration: underline">"TPMT Results"</span></a><span class="cql-object"> T</span>
        <br/>
        <div class="p-l-10">
          <span class="cql-keyword">where </span><span class="cql-object">T</span><span class="cql-generic">.</span><span class="cql-literal">value </span><span class="cql-operator">= </span><!--
          --><span class="cql-keyword">Code </span><span class="cql-generic">{ </span> <span class="cql-literal">code</span><span class="cql-generic">: </span><!--
          --><span class="cql-string">'LA10317-8' </span><span class="cql-literal">system</span><span class="cql-generic">: </span><span class="cql-string">"http://loinc.org" </span><!--
          --><span class="cql-literal">display</span><span class="cql-generic">: </span><span class="cql-string">'Intermediate Metabolizer' </span><span class="cql-generic">}</span>
        </div>
      </div>
    </div>
  </p>

  <p>
    And for a Normal metabolizer TPMT gene product metabolic activity interpretation observation...<br/>
    <div class="code">
      <span class="cql-keyword">define </span><span class="cql-string" id="Normal">"Normal Status"</span><span class="cql-generic">:</span>
      <br/>
      <div class="p-l-10">
        <span class="cql-string">"TPMT Results"</span><span class="cql-object"> T</span>
        <br/>
        <div class="p-l-10">
          <span class="cql-keyword">where </span><span class="cql-object">T</span><span class="cql-generic">.</span><span class="cql-literal">value </span><span class="cql-operator">= </span><!--
          --><span class="cql-keyword">Code </span><span class="cql-generic">{ </span> <span class="cql-literal">code</span><span class="cql-generic">: </span><!--
          --><span class="cql-string">'LA25391-6' </span><span class="cql-literal">system</span><span class="cql-generic">: </span><span class="cql-string">"http://loinc.org" </span><!--
          --><span class="cql-literal">display</span><span class="cql-generic">: </span><span class="cql-string">'Normal Metabolizer' </span><span class="cql-generic">}</span>
        </div>
      </div>
    </div>
  </p>

  <p>
    And for a Poor metabolizer TPMT gene product metabolic activity interpretation observation...<br/>
    <div class="code">
      <span class="cql-keyword">define </span><span class="cql-string" id="Poor">"Poor Status"</span><span class="cql-generic">:</span>
      <br/>
      <div class="p-l-10">
        <span class="cql-string">"TPMT Results"</span><span class="cql-object"> T</span>
        <br/>
        <div class="p-l-10">
          <span class="cql-keyword">where </span><span class="cql-object">T</span><span class="cql-generic">.</span><span class="cql-literal">value </span><span class="cql-operator">= </span><!--
          --><span class="cql-keyword">Code </span><span class="cql-generic">{ </span> <span class="cql-literal">code</span><span class="cql-generic">: </span><!--
          --><span class="cql-string">'LA9657-3' </span><span class="cql-literal">system</span><span class="cql-generic">: </span><span class="cql-string">"http://loinc.org" </span><!--
          --><span class="cql-literal">display</span><span class="cql-generic">: </span><span class="cql-string">'Poor Metabolizer' </span><span class="cql-generic">}</span>
        </div>
      </div>
    </div>
  </p>

  <a name="g037"/>
  <h5>Additional Use Cases</h5>

  <p>The following list of proposed LOINC codes are provided to complete the other HLA B alleles and Pharmacogenetic test results that are
    covered by Q7 of Survey #3 and Q6 of Survey #4 of the CPIC Delphi Survey, respectively.  There is minimal additional effort in defining the
    LOINC codes for these additional use cases. The potential benefit is that it may allow CDS implementers and labs to easily incorporate CDS rules
    similar to abacavir and azathioprine for these additional use cases providing benefit to additional patient populations.</p>

  <p>Additional HLA-B Allele Findings from Q7 of the CPIC Delphi Survey #3 results.<br/>
    <ol>
      <li>Carbamazepine and phenytoin cutaneous reactions<br/> 57979-7: HLA-B*15:02 [Presence]</li>
      <li>Allopurinol cutaneous reactions <br/> 79711-8: HLA-B*58:01 [Presence]</li>

    </ol>
  </p>

  <p>Additional pharmacogenetic test result codes applying to the drug metabolizing enzyme genes (Q4 of the CPIC Delphi Survey #4 results:
    includes CYP2C9, CYP2C19, CYP2D6, CYP3A5, DPYD, and UGT1A1).  For example, the DPYD gene product metabolic activity interpretation code would
    be modeled after the code that is proposed for TPMT in the section above.<br/>
    <ol>
      <li>79719-1: DPYD gene product metabolic activity interpretation</li>
    </ol>
  </p>

  <a name="establishing"/>
  <h4>Establishing the Clinical Decision Support Rules</h4>
  <p>
    Information provided below is intended to help implementers establish specific forms of clinical decision support.
    However, this implementation guide should not be considered a guideline. Each site must appropriately review and validate
    its content prior to implementation. We hope that this guide serves as a useful template but we encourage implementers to
    edit our proposed text and rules as they deem appropriate.
    <br/><br/>
    <em>
      The following clinical decision support rules will identify conditions that, once triggered, will cause some action by the system. In this case,
      the action is to alert the clinician(s) responsible for the patient. The conditions below are provided in plain text and CQL. In addition, each condition
      and the associated guidance is described in a DecisionSupportRule resource distributed with this IG.
    </em>
  </p>

  <a name="g041"/>
  <h5>Abacavir Order Phamacogenomic Support</h5>
  <p>For each order of Abacavir, the system should intervene if:
    <ul>
      <li>
        The patient has not been administered a genotype test
        <div class="code">
          <span class="cql-keyword">define </span><span class="cql-string">"Needs Genotype Test"</span><span class="cql-generic">:</span>
          <br/>
          <div class="p-l-10">
            <span class="cql-keyword">not exists </span><span class="cql-generic">(</span><a href="#ObservedAllele"><span class="cql-interface" style="text-decoration: underline">"Observed HLA-B*57:01 Allele"</span></a><span class="cql-generic">)</span>
          </div>
        </div>
        <a href="decisionsupportrule-abacavir-need-test.xml.html">Decision Support Rule</a>
      </li>
      <li>
        There are two or more conflicting statuses of the observed HLA-B*57:01 allele from a patient's genotype test results
        <div class="code">
          <span class="cql-keyword">define </span><span class="cql-string">"Should Notify User of Conflicting Results"</span><span class="cql-generic">:</span>
          <br/>
          <div class="p-l-10">
            <span class="cql-keyword">exists </span><span class="cql-generic">(</span><a href="#PositiveHyper"><span class="cql-interface" style="text-decoration: underline">"Positive Status for Hypersensitivity"</span></a><span class="cql-generic">)</span>
            <div class="p-l-10">
              <span class="cql-keyword">and exists </span><span class="cql-generic">(</span><a href="#NegativeHyper"><span class="cql-interface" style="text-decoration: underline">"Negative Status for Hypersensitivity"</span></a><span class="cql-generic">)</span>
            </div>
          </div>
        </div>
        <a href="decisionsupportrule-abacavir-conflicting-statuses.xml.html">Decision Support Rule</a>
      </li>
      <li>
        There is a positive status in the patient's genotype test indicating a hypersensitivity to Abacavir
        <div class="code">
          <span class="cql-keyword">define </span><span class="cql-string">"Should Notify User of Patient's Hypersensitivity"</span><span class="cql-generic">:</span>
          <br/>
          <div class="p-l-10">
            <span class="cql-keyword">exists </span><span class="cql-generic">(</span><a href="#PositiveHyper"><span class="cql-interface" style="text-decoration: underline">"Positive Status for Hypersensitivity"</span></a><span class="cql-generic">)</span>
            <div class="p-l-10">
              <span class="cql-keyword">and not exists </span><span class="cql-generic">(</span><a href="#NegativeHyper"><span class="cql-interface" style="text-decoration: underline">"Negative Status for Hypersensitivity"</span></a><span class="cql-generic">)</span>
            </div>
          </div>
        </div>
        <a href="decisionsupportrule-abacavir-hypersensitivity-positive.xml.html">Decision Support Rule</a>
      </li>
    </ul>
  </p>
  
  <a name="g042"/>
  <h5>Azathioprine Order Phamacogenomic Support</h5>
  <p>For each order of Azathioprine, the system should intervene if:
    <ul>
      <li>
        The patient has not been administered a TPMT genotype test.
        <div class="code">
          <span class="cql-keyword">define </span><span class="cql-string">"Needs TPMT Test"</span><span class="cql-generic">:</span>
          <br/>
          <div class="p-l-10">
            <span class="cql-keyword">not exists </span><span class="cql-generic">(</span><a href="#Results"><span class="cql-interface" style="text-decoration:underline;">"TPMT Results"</span></a><span class="cql-generic">)</span>
          </div>
        </div>
        <a href="decisionsupportrule-azathioprine-need-test.xml.html">Decision Support Rule</a>
      </li>
      <li>
        There are two or more conflicting TPMT results for a patient.
        <div class="code">
          <span class="cql-keyword">define </span><span class="cql-string" id="ConflictingTPMT">"Should Notify User of Conflicting Results"</span><span class="cql-generic">:</span>
          <br/>
          <div class="p-l-10">
            <span class="cql-keyword">exists </span><span class="cql-generic">(</span><a href="#Normal"><span class="cql-interface" style="text-decoration:underline;">"Normal Status"</span></a><span class="cql-generic">)</span>
            <div class="p-l-10">
              <span class="cql-keyword">and exists </span><span class="cql-generic">(</span><a href="#Intermediate"><span class="cql-interface" style="text-decoration:underline;">"Intermediate Status"</span></a><span class="cql-generic">)</span>
              <div class="p-l-10">
                <span class="cql-keyword">or exists </span><span class="cql-generic">(</span><a href="#Poor"><span class="cql-interface" style="text-decoration:underline;">"Poor Status"</span></a><span class="cql-generic">)</span>
              </div>
            </div>
          </div>
        </div>
        <a href="decisionsupportrule-azathioprine-conflicting-statuses.xml.html">Decision Support Rule</a>
      </li>
      <li>
        The patient is at increased risk of myelotoxicity due to intermediate status on TPMT genotype test
        <div class="code">
          <span class="cql-keyword">define </span><span class="cql-string">"Should Notify User to Lower Dosage"</span><span class="cql-generic">:</span>
          <br/>
          <div class="p-l-10">
            <span class="cql-keyword">exists </span><span class="cql-generic">(</span><a href="#Intermediate"><span class="cql-interface" style="text-decoration:underline;">"Intermediate Status"</span></a><span class="cql-generic">)</span>
            <div class="p-l-10">
              <span class="cql-keyword">and not exists </span><span class="cql-generic">(</span><a href="#ConflictingTPMT"><span class="cql-interface" style="text-decoration:underline;">"Should Notify User of Conflicting Statuses"</span></a><span class="cql-generic">)</span>
            </div>
          </div>
        </div>
        <a href="decisionsupportrule-azathioprine-lower-dosage.xml.html">Decision Support Rule</a>
      </li>
      <li>
        The patient is at very high risk of myelotoxicity due to poor status found in the latest TPMT genotype test
        <div class="code">
          <span class="cql-keyword">define </span><span class="cql-string">"Should Notify User of Poor Status in Most Recent Record"</span><span class="cql-generic">:</span>
          <br/>
          <div class="p-l-10">
            <span class="cql-keyword">exists </span><span class="cql-generic">(</span><span class="cql-string">"First Record Poor Status"</span><span class="cql-generic">)</span>
          </div>
          <br/>
          <span class="cql-keyword">define </span><span class="cql-string">"First Record Poor Status"</span><span class="cql-generic">:</span>
          <br/>
          <div class="p-l-10">
            <a href="#Results"><span class="cql-interface" style="text-decoration:underline;">"TPMT Results"</span></a><span class="cql-object"> T</span>
            <br/>
            <div class="p-l-10">
              <span class="cql-keyword">where </span><span class="cql-keyword">First</span><span class="cql-generic">(</span><span class="cql-object">T</span><span class="cql-generic">.</span><span class="cql-literal">value </span><span class="cql-operator">= </span><!--
              --><span class="cql-keyword">Code </span><span class="cql-generic">{ </span> <span class="cql-literal">code</span><span class="cql-generic">: </span><!--
              --><span class="cql-string">'LA9657-3' </span><span class="cql-literal">system</span><span class="cql-generic">: </span><span class="cql-string">"http://loinc.org" </span><!--
              --><span class="cql-literal">display</span><span class="cql-generic">: </span><span class="cql-string">'Poor Metabolizer' </span><span class="cql-generic">})</span>
              <div class="p-l-10">
                <span class="cql-keyword">sort by </span><span class="cql-object">T</span><span class="cql-generic">.</span><span class="cql-class">effectiveDateTime</span>
              </div>
            </div>
          </div>
        </div>
        <a href="decisionsupportrule-azathioprine-poor-status.xml.html">Decision Support Rule</a>
      </li>
    </ul>
  </p>
  
  <a name="content-index"/>
  <h4>Content Index</h4>
  
  <p>The following resources formalize the description of the logic and behavior defined by this implementation guide.</p>
  
  <table class="list">
	  <thead>
		  <tr><th>Resource</th><th>Type</th><th>Description</th></tr>
	  </thead>
	  <tbody>
		  <tr><td><a href="decisionsupportrule-abacavir-conflicting-statuses.xml">Abacavir Conflicting Statuses</a></td><td><a href="[%fhir-path%]decisionsupportrule.html">Decision Support Rule</a></td><td>Detects conflicting statuses for Abacavir Sensitivity tests and notifies appropriately</td></tr>
		  <tr><td><a href="decisionsupportrule-abacavir-hypersensitivity-positive.xml">Abacavir Hypersensitivity</a></td><td><a href="[%fhir-path%]decisionsupportrule.html">Decision Support Rule</a></td><td>Detects positive status for Abacavir Sensitivity and notifies appropriately</td></tr>
		  <tr><td><a href="decisionsupportrule-abacavir-need-test.xml">Abacavir Sensitivity Test Recommended</a></td><td><a href="[%fhir-path%]decisionsupportrule.html">Decision Support Rule</a></td><td>Detects no results for Abacavir Sensitivity and notifies appropriately</td></tr>
		  <tr><td><a href="decisionsupportrule-azathioprine-conflicting-statuses.xml">Azathioprine Conflicting Statuses</a></td><td><a href="[%fhir-path%]decisionsupportrule.html">Decision Support Rule</a></td><td>Detects conflicting statuses for Azathioprine Sensitivity tests and notifies appropriately</td></tr>
		  <tr><td><a href="decisionsupportrule-azathioprine-lower-dosage.xml">Azathioprine Lower Dosage</a></td><td><a href="[%fhir-path%]decisionsupportrule.html">Decision Support Rule</a></td><td>Detects that a lower dosage of Azathioprine may be appropriate</td></tr>
		  <tr><td><a href="decisionsupportrule-azathioprine-need-test.xml">Azathioprine Test Recommended</a></td><td><a href="[%fhir-path%]decisionsupportrule.html">Decision Support Rule</a></td><td>Detects no results for Azathioprine Sensitivity and notifies appropriately</td></tr>
		  <tr><td><a href="decisionsupportrule-azathioprine-poor-status.xml">Azathioprine Poor Status</a></td><td><a href="[%fhir-path%]decisionsupportrule.html">Decision Support Rule</a></td><td>Detects potentially poor metabolism for Azathioprine and notifies appropriately</td></tr>
		  <tr><td><a href="library-abacavir-hypersensitivity-cds-logic.xml">Abacavir Hypersensitivity Library</a></td><td><a href="[%fhir-path%]library.html">Library</a></td><td>The distributable library for Abacavir Sensitivity logic used by the Abacavir Sensitivity decision support rules.</td></tr>
		  <tr><td><a href="library-azathioprine-myelotoxicity-cds-logic.xml">Abacavir Myelotoxicity Library</a></td><td><a href="[%fhir-path%]library.html">Library</a></td><td>The distributable library for Azathioprine Myelotoxicity logic used by the Azathioprine Myelotoxicity decision support rules.</td></tr>
		  <tr><td><a href="AbacavirOrderingPharmacogenomicSupport.cql">Abacavir Sensitivity CQL</a></td><td>CQL Source</td><td>For reference, the complete CQL content of the Abacavir Hypersensitivity Library</td></tr>
		  <tr><td><a href="AzathioprineOrderingPharmacogenomicSupport.cql">Azathioprine Myelotoxicity CQL</a></td><td>CQL Source</td><td>For reference, the complete CQL content of the Azathioprine Myelotoxicity Library</td></tr>
	  </tbody>
  </table>
  
  <a name="references"/>
  <h4>References</h4>
  <p>
    Bell, Gillian C., Kristine R. Crews, Mark R. Wilkinson, Cyrine E. Haidar, J. Kevin Hicks, Donald K. Baker, Nancy M. Kornegay et al.
    "Development and use of active clinical decision support for preemptive pharmacogenomics." Journal of the American Medical Informatics
    Association 21, no. e1 (2014): e93-e99.
  </p>

  <p>
    Martin, M. A., et al. "Clinical pharmacogenetics implementation consortium guidelines for HLA-B genotype and abacavir dosing."
    Clinical Pharmacology &amp; Therapeutics 91.4 (2012): 734-738.
  </p>

  <p>
    Peterson JF, Bowton E, Field JR, Beller M, Mitchell J, Schildcrout J, Gregg W, Johnson K, Jirjis JN, Roden DM, Pulley JM, Denny JC.
    "Electronic health record design and implementation for pharmacogenomics: a local perspective." Genet Med. 2013 Oct;15(10):833-41.
  </p>

  <p>
    CPIC Term Standardization Project: <a href="https://www.pharmgkb.org/page/cpicTermProject">https://www.pharmgkb.org/page/cpicTermProject</a>
  </p>

  <p>
    SGE Marsh, ED Albert, WF Bodmer, RE Bontrop, B Dupont, HA Erlich, M Fernandez-Vina, DE Geraghty, R Holdsworth, CK Hurley, M Lau, KW Lee,
    B Mach, WR Mayr, M Maiers, CR Muller, P Parham, EW Petersdorf, T Sasazuki, JL Strominger, A Svejgaard, PI Terasaki, JM Tiercy, J Trowsdale.
    "Nomenclature for factors of the HLA system, 2010." Tissue Antigens (2010) 75:291-455
  </p>

</div>

[%file newfooter%]
</body>
</html>
